export const getPubResult = {
	id: "14b26f27-ffc0-4909-8ba9-c7788cffba8e",
	slug: "result-venom-capsids",
	title: "Identification of capsid-like proteins in venomous and parasitic animals",
	htmlTitle: "Identification of capsid-like proteins in venomous and parasitic animals",
	description:
		"Inspired by wasps co-opting viral capsids to deliver genes to the caterpillars they parasitize, we looked for capsid-like proteins in other species. We found capsid homologs in ticks and other parasites, suggesting this phenomenon could be wider spread than previously known.",
	htmlDescription:
		"Inspired by wasps co-opting viral capsids to deliver genes to the caterpillars they parasitize, we looked for capsid-like proteins in other species. We found capsid homologs in ticks and other parasites, suggesting this phenomenon could be wider spread than previously known.",
	avatar: "https://assets.pubpub.org/yhsu8e81/Arcadia_Pub type_Preview_Result (1)-71721329283073.png",
	customPublishedAt: null,
	doi: "10.57844/arcadia-14b2-6f27",
	labels: null,
	downloads: null,
	metadata: null,
	viewHash: "rxo0ype3",
	editHash: "rdelhu4v",
	reviewHash: "ejaogp82",
	commentHash: "jcnu516p",
	draftId: "8fae50d7-1652-45b0-9d0f-39091d1ba504",
	communityId: "1a71ef4d-f6fe-40d3-8379-42fa2141db58",
	crossrefDepositRecordId: "3f0ca97c-21b5-4850-8105-f41b01f1aad3",
	scopeSummaryId: "23d01957-ccdf-4986-9f31-6103590a9b4d",
	createdAt: "2024-03-06T19:05:39.597Z",
	updatedAt: "2024-09-27T17:44:49.715Z",
	draft: {
		id: "8fae50d7-1652-45b0-9d0f-39091d1ba504",
		latestKeyAt: "2024-08-09T20:31:24.234Z",
		firebasePath: "drafts/draft-8fae50d7-1652-45b0-9d0f-39091d1ba504",
		createdAt: "2024-03-06T19:05:39.594Z",
		updatedAt: "2024-09-26T19:26:28.995Z",
	},
	members: [
		{
			id: "aeda7fc1-123a-4057-94f2-14dfd7ef10ef",
			permissions: "manage",
			isOwner: true,
			subscribedToActivityDigest: false,
			userId: "15b7916f-ce5a-4125-8eaf-cb2d9cd689a6",
			pubId: "14b26f27-ffc0-4909-8ba9-c7788cffba8e",
			collectionId: null,
			communityId: null,
			organizationId: null,
			createdAt: "2024-03-06T19:05:39.636Z",
			updatedAt: "2024-03-06T19:05:39.636Z",
			user: {
				id: "15b7916f-ce5a-4125-8eaf-cb2d9cd689a6",
				firstName: "Robert",
				lastName: "Roth",
				fullName: "Robert Roth",
				avatar: "https://assets.pubpub.org/bcncv9cu/41682358423746.jpg",
				slug: "robert-roth",
				initials: "RR",
				title: "Publishing Project Manager at Arcadia Science",
				orcid: "0000-0003-2604-1866",
			},
		},
		{
			id: "3b6aa793-8d79-4cbb-8b02-79df3ce57467",
			permissions: "admin",
			isOwner: null,
			subscribedToActivityDigest: false,
			userId: "30bf0914-8367-4ae4-ab96-536b98dc7a19",
			pubId: "14b26f27-ffc0-4909-8ba9-c7788cffba8e",
			collectionId: null,
			communityId: null,
			organizationId: null,
			createdAt: "2024-07-26T20:01:19.552Z",
			updatedAt: "2024-08-01T18:20:18.900Z",
			user: {
				id: "30bf0914-8367-4ae4-ab96-536b98dc7a19",
				firstName: "Adair L.",
				lastName: "Borges",
				fullName: "Adair L. Borges",
				avatar: "https://assets.pubpub.org/ksstjecd/41668532775908.jpg",
				slug: "adair-borges",
				initials: "AB",
				title: "Adair",
				orcid: "0000-0002-1477-7908",
			},
		},
		{
			id: "ade8dc73-3cdf-40c5-a8eb-b71c5971bea0",
			permissions: "view",
			isOwner: null,
			subscribedToActivityDigest: false,
			userId: "25a86dff-6ad5-473d-ae1c-e8f46d9f24b9",
			pubId: "14b26f27-ffc0-4909-8ba9-c7788cffba8e",
			collectionId: null,
			communityId: null,
			organizationId: null,
			createdAt: "2024-08-09T21:13:23.922Z",
			updatedAt: "2024-08-09T21:13:23.922Z",
			user: {
				id: "25a86dff-6ad5-473d-ae1c-e8f46d9f24b9",
				firstName: "Megan L.",
				lastName: "Hochstrasser",
				fullName: "Megan L. Hochstrasser",
				avatar: "https://assets.pubpub.org/7gf037kv/71647900864012.jpg",
				slug: "megan-hochstrasser-2",
				initials: "MH",
				title: "Director of Publishing",
				orcid: "0000-0002-4404-078X",
			},
		},
	],
} as const;

export const getPubText = {
	type: "doc",
	attrs: {
		meta: {},
		suggestionId: null,
		suggestionTimestamp: null,
		suggestionUserId: null,
		suggestionDiscussionId: null,
		suggestionKind: null,
		suggestionOriginalAttrs: null,
	},
	content: [
		{
			type: "heading",
			attrs: {
				level: 1,
				fixedId: "",
				id: "purpose",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Purpose",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "n33ucq2qaha",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "The development of AAV capsids for therapeutic gene delivery has exploded in popularity over the past few years. However, humans aren’t the first or only species using viral capsids for gene delivery — wasps evolved this tactic over 100 million years ago. Parasitoid wasps that lay eggs inside arthropod hosts have co-opted ancient viruses for gene delivery to manipulate multiple aspects of the host’s biology, thereby increasing the probability of survival of the wasp larvae ",
				},
				{
					type: "citation",
					attrs: {
						id: "n67l65xpyip",
						href: null,
						value: "https://doi.org/10.1016/j.virusres.2006.01.001",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "citation",
					attrs: {
						id: "n2piklt9xg9",
						href: null,
						value: "https://doi.org/10.1016/j.tim.2004.10.004",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ". ",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nucymbbl1uc",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "We wondered if venomous species that bite humans have also evolved to use viral capsids to deliver molecules that manipulate their hosts. We used a multi-pronged sequence- and structure-based search for viral capsids across venomous species and found evidence for endogenized viral capsids, most notably in ticks. Though we cast a broad net in this effort, we were most interested in finding novel nucleic acid delivery systems in species that deliver cargo to humans, as these would be the most useful therapeutic modalities. Here, we focused on finding endogenized capsid proteins, and our follow-up work has focused on developing methods to specifically identify potential nucleic acid cargos in parasites. More to come on this in a follow-up pub!",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nepq27n6yik",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "These findings may be useful to other scientists interested in the domestication of viral genes, especially in the context of parasitism.",
				},
			],
		},
		{
			type: "bullet_list",
			attrs: {
				id: "nxagub6a2yz",
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "list_item",
					attrs: {
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "paragraph",
							attrs: {
								id: "njuywwe9ie7",
								class: null,
								textAlign: null,
								rtl: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "text",
									marks: [
										{
											type: "strong",
										},
									],
									text: "Data ",
								},
								{
									type: "text",
									text: "from our capsid HMM search and our preHGT run to find putatively endogenized viral capsids are available on ",
								},
								{
									type: "text",
									marks: [
										{
											type: "link",
											attrs: {
												href: "https://zenodo.org/records/12775362",
												title: null,
												target: "_blank",
												pubEdgeId: null,
											},
										},
									],
									text: "Zenodo",
								},
								{
									type: "text",
									text: ", as is our follow-up work looking into individual hits with BLAST searches of tick salivary transcriptome, genomic neighborhood analysis, and Foldseek analysis.",
								},
							],
						},
					],
				},
				{
					type: "list_item",
					attrs: {
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "paragraph",
							attrs: {
								id: "n3ed7v01mlb",
								class: null,
								textAlign: null,
								rtl: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "text",
									marks: [
										{
											type: "strong",
										},
									],
									text: "Data ",
								},
								{
									type: "text",
									text: "from our ProteinCartography runs, including structures of all tick and viral capsid proteins, the configuration file, and all outputs, are available on ",
								},
								{
									type: "text",
									marks: [
										{
											type: "link",
											attrs: {
												href: "https://zenodo.org/records/12796464",
												title: null,
												target: "_blank",
												pubEdgeId: null,
											},
										},
									],
									text: "Zenodo",
								},
								{
									type: "text",
									text: ". We performed ProteinCartography analysis using version 0.4.2 of the pipeline, found in ",
								},
								{
									type: "text",
									marks: [
										{
											type: "link",
											attrs: {
												href: "https://github.com/Arcadia-Science/ProteinCartography/tree/v0.4.2",
												title: null,
												target: "_blank",
												pubEdgeId: null,
											},
										},
									],
									text: "this GitHub repository",
								},
								{
									type: "text",
									text: ".",
								},
							],
						},
					],
				},
				{
					type: "list_item",
					attrs: {
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "paragraph",
							attrs: {
								id: "n4teo6c9gk4",
								class: null,
								textAlign: null,
								rtl: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "text",
									text: "All associated ",
								},
								{
									type: "text",
									marks: [
										{
											type: "strong",
										},
									],
									text: "code",
								},
								{
									type: "text",
									text: " is available in this ",
								},
								{
									type: "text",
									marks: [
										{
											type: "link",
											attrs: {
												href: "https://github.com/Arcadia-Science/capsids/tree/pubv1.0",
												title: null,
												target: "_blank",
												pubEdgeId: null,
											},
										},
									],
									text: "GitHub repository",
								},
								{
									type: "text",
									text: ".",
								},
							],
						},
					],
				},
			],
		},
		{
			type: "blockquote",
			attrs: {
				id: "nijip1ieibq",
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "paragraph",
					attrs: {
						id: "nkjq02met0t",
						class: null,
						textAlign: null,
						rtl: null,
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "text",
							text: "Share your thoughts!",
						},
					],
				},
				{
					type: "paragraph",
					attrs: {
						id: "n7u3k403rps",
						class: null,
						textAlign: null,
						rtl: null,
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
								{
									type: "link",
									attrs: {
										href: "https://bit.ly/3xo7dkm",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "Watch a video tutorial",
						},
						{
							type: "text",
							text: " on making a PubPub account and commenting. Feel free to add line-by-line comments anywhere within this text, provide overall feedback by commenting in the box at the bottom of the page, or post about this work on social media. Please make all feedback public so other readers can benefit from the discussion.",
						},
					],
				},
			],
		},
		{
			type: "heading",
			attrs: {
				level: 1,
				fixedId: "",
				id: "weve-put-this-effort-on-ice",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "We’ve put this effort on ice! 🧊",
				},
			],
		},
		{
			type: "blockquote",
			attrs: {
				id: "npj96iygw1a",
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "paragraph",
					attrs: {
						id: "n9q55683k02",
						class: null,
						textAlign: null,
						rtl: null,
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
							],
							text: "#DeadEnd",
						},
					],
				},
				{
					type: "paragraph",
					attrs: {
						id: "ndkwbii73ia",
						class: null,
						textAlign: null,
						rtl: null,
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "text",
							text: "We found intriguing capsid-like proteins across several parasitic species. To advance, we'd need to figure out what cargo the putative capsids deliver, if any. In a follow-up study, we tried to computationally identify packaged nucleic acids in parasites, as we’re most interested in therapeutic nucleic acid delivery, but our results weren’t sufficiently compelling to warrant further investment.",
						},
					],
				},
				{
					type: "paragraph",
					attrs: {
						id: "n4qmfzify47",
						class: null,
						textAlign: null,
						rtl: null,
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "text",
							marks: [
								{
									type: "link",
									attrs: {
										href: "https://research.arcadiascience.com/icebox",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
								{
									type: "sup",
								},
							],
							text: "Learn more",
						},
						{
							type: "text",
							marks: [
								{
									type: "sup",
								},
							],
							text: " about the Icebox and the different reasons we ice projects.",
						},
					],
				},
			],
		},
		{
			type: "heading",
			attrs: {
				level: 1,
				fixedId: "",
				id: "background-and-goals",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Background and goals",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nobsrjekjzw",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Capsid-based delivery of therapeutics represents a growing modality of interest across the biotech/pharma industry. The focus of most recent efforts has been on a specific class of capsids from adenovirus-associated viruses (AAVs). AAV-based therapies have been approved for use in humans, and researchers are engineering and applying these vectors for a variety of indications ",
				},
				{
					type: "citation",
					attrs: {
						id: "n05riw93ovg",
						href: null,
						value: "https://doi.org/10.1038/s41573-019-0012-9",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ". However, co-opting viral capsids for gene delivery is an ancient innovation, dating back over 100 million years.",
				},
			],
		},
		{
			type: "image",
			attrs: {
				id: "njp74m066hd",
				url: "https://assets.pubpub.org/39g5lkfx/Fig1_Parasitoid-wasp-GA-31721329603673.png",
				size: 85,
				align: "center",
				caption:
					'<p id="n9eqg1k4tzp" data-reactroot=""><strong>Braconid parasitoid wasps use domesticated viruses to protect their eggs from the host immune system</strong>.</p><p id="r6807458188" data-reactroot="">Braconid parasitoid wasps like <em>Microplitis demolitor</em> lay their eggs in living lepidopteran hosts, where the larvae consume the host as a food source throughout development. To protect their eggs from the caterpillar immune response, the wasps evolved a fascinating strategy of deploying an ancient endogenized betanudivirus (called a bracovirus) as a gene delivery agent. The wasps inject viral particles containing wasp DNA alongside their eggs, and prolonged expression of these genes in the caterpillar alters the host immune system to protect the developing larvae.</p><p id="r6313380962" data-reactroot="">mya: Million years ago</p>',
				altText: "",
				hideLabel: false,
				fullResolution: true,
				href: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
		},
		{
			type: "paragraph",
			attrs: {
				id: "njbuy8jyy9p",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Scientists uncovered this innovation by studying parasitoid wasps, which use viral capsids derived from ancient polydnaviruses to deliver genes to their caterpillar hosts (",
				},
				{
					type: "reference",
					attrs: {
						id: "nf4s8uih8r2",
						targetId: "njp74m066hd",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ") ",
				},
				{
					type: "citation",
					attrs: {
						id: "nq3mf22frzr",
						href: null,
						value: "https://doi.org/10.1038/s42003-020-01623-8",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "citation",
					attrs: {
						id: "n271lwt830j",
						href: null,
						value: "https://doi.org/10.1016/j.coviro.2017.07.002",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ". While there is an enormous diversity of ecologies and lifestyles across parasitoid wasps, many parasitize other arthropods, especially lepidopterans. These wasps inject their eggs into caterpillars, and the developing larvae use the host as a source of food. To help protect the eggs, the parasitoid wasps co-inject viral-like particles that manipulate the caterpillar’s biology, including its immune system ",
				},
				{
					type: "citation",
					attrs: {
						id: "n4h9efgzc3i",
						href: null,
						value: "https://doi.org/10.3389/fimmu.2022.877027",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ". The injected packages are made up of viral capsids filled with wasp DNA that helps to protect the wasps’s larvae. Once injected into the host, wasp DNA is integrated into the host genome to drive expression of host-manipulating factors, such as immune-dampening proteins.",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "n0n6b4uf95d",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Wasps have co-opted distinct types of polydnaviruses multiple times ",
				},
				{
					type: "citation",
					attrs: {
						id: "n1pek8zaqhp",
						href: null,
						value: "https://doi.org/10.1098/rstb.2013.0051",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ", with different wasp lineages harboring distinct endogenized viruses that they deploy for host manipulation. Of these, some of the best-studied are the bracoviruses found in the genomes of braconid wasps like ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Microplitis",
				},
				{
					type: "text",
					text: " and ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Cotesia. ",
				},
				{
					type: "text",
					text: "Although the bracovirus capsid proteins play a key role in delivery of nucleic acid cargo to host species, studies on bracovirus biology show key roles for other virus-derived machinery in the process — especially packaging machinery and envelope proteins.",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nzutgw5cgsj",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Our goal in this pilot was to explore whether other venomous species have similarly evolved the ability to use viral capsids for molecular delivery. Though we searched for endogenized capsids of any sort, we’d hoped to identify capsids that deliver genes to a human host. If we can find such capsids, they might serve as new starting points for developing gene delivery systems.",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nkwak1lq67y",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "We used a combination of computational approaches to scan for the presence of capsid-like proteins across venoms, including sequence-, HGT-, and structure-based searches. We ultimately found capsid-like genes in the venom-like saliva of ticks as well as in the genomes of other parasitic species, including parasitic nematodes (helminths), mosquitoes, and aphids. Jump straight to ",
				},
				{
					type: "text",
					marks: [
						{
							type: "link",
							attrs: {
								href: "#the-results",
								title: null,
								target: null,
								pubEdgeId: null,
							},
						},
					],
					text: "these results",
				},
				{
					type: "text",
					text: " or read on to learn the details of our approach.",
				},
			],
		},
		{
			type: "heading",
			attrs: {
				level: 1,
				fixedId: "",
				id: "the-approach",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "The approach",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nxan7acu8f8",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "To identify capsid-like proteins in venomous species, we applied two strategies: a ",
				},
				{
					type: "text",
					marks: [
						{
							type: "strong",
						},
					],
					text: "sequence homology-based search",
				},
				{
					type: "text",
					text: " for capsids in venom gland transcriptomes (HMM-based detection) and our ",
				},
				{
					type: "text",
					marks: [
						{
							type: "strong",
						},
					],
					text: "horizontal gene transfer (HGT) detection pipeline",
				},
				{
					type: "text",
					text: " to detect genes of putative viral origin in genomes of venomous species (preHGT). We then followed up on the most promising hits by searching for structural homologs across the AlphaFold database using Foldseek. Last, we experimented with a structural clustering-based approach to identify capsid-like structures in salivary transcriptomes but didn’t pursue this approach extensively as it wasn’t scalable and results were hard to interpret.",
				},
			],
		},
		{
			type: "heading",
			attrs: {
				level: 2,
				fixedId: "",
				id: "an-hmm-based-search-of-venom-transcriptomes-for-viral-capsids",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "An HMM-based search of venom transcriptomes for viral capsids",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "n9ywpb2u23u",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "We first performed a sequence-based search for capsid-like proteins in venom transcriptomes using hidden Markov models (HMMs) of viral capsids. A curated database of viral protein families and associated HMM profiles for each family is available for download from the ",
				},
				{
					type: "text",
					marks: [
						{
							type: "link",
							attrs: {
								href: "https://vogdb.org/",
								title: null,
								target: "_blank",
								pubEdgeId: null,
							},
						},
					],
					text: "viral orthologous groups (VOGs) database",
				},
				{
					type: "text",
					text: " (version vog217). In addition, because bracovirus capsids aren't included in the VOG database, we also generated a custom HMM profile to include in searches based on a PSI-BLAST search results using the ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Microplitis demolitor",
				},
				{
					type: "text",
					text: " major bracovirus capsid protein vp39 as a seed sequence.",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nbfinv30fl8",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "We then used these HMMs to search against 156 transcriptomes from 135 species using HMMER3 ",
				},
				{
					type: "citation",
					attrs: {
						id: "ncs4jrwy8dn",
						href: null,
						value: "https://doi.org/10.1093/nar/gkr367",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ". To maximize our ability to find distant homologs, we initially returned all hits from the HMMER search with no E-value cutoff and followed up with manual filtering steps.",
				},
			],
		},
		{
			type: "blockquote",
			attrs: {
				id: "njkopr46hh7",
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "paragraph",
					attrs: {
						id: "nir143deub7",
						class: null,
						textAlign: null,
						rtl: null,
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "text",
							text: "You can find the ",
						},
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
							],
							text: "list of species",
						},
						{
							type: "text",
							text: " included in this database, along with all ",
						},
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
							],
							text: "scripts, metadata, and results",
						},
						{
							type: "text",
							text: ", in ",
						},
						{
							type: "text",
							marks: [
								{
									type: "link",
									attrs: {
										href: "https://github.com/Arcadia-Science/capsids/tree/pubv1.0",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "this GitHub repository",
						},
						{
							type: "text",
							text: ".",
						},
					],
				},
			],
		},
		{
			type: "heading",
			attrs: {
				level: 3,
				fixedId: "",
				id: "filtering-results-from-the-hmm-based-search",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Filtering results from the HMM-based search ",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nd4c0abur24",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "We first filtered results to include only hits from a subset of VOGs containing the word “capsid,” but not “capsid maturation protease” or “capsid assembly protease,” in their annotation, as we found these VOGs don’t encode actual capsid proteins themselves, and were driving a lot of spurious hits to animal proteases. Next, we manually curated the output file in an attempt to further filter out false positives. This analysis suffered from three major sources of false positives: hits from non-endogenized viruses that are part of the venom virome, hits to venom proteins that don't appear to be endogenized capsids but share a common domain with the capsid HMM (for instance, a protease domain or an immunoglobulin domain), and spurious hits to the animal proteome due the fact we didn't use any E-value cutoff for our initial HMMER search. ",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nhecxvatzqb",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "To identify false positives, we performed BLASTp on the top 210 hits from the HMMER3 search (E-value 1.20 × 10",
				},
				{
					type: "text",
					marks: [
						{
							type: "sup",
						},
					],
					text: "−81",
				},
				{
					type: "text",
					text: "–0.0026). We performed these searches using the NCBI web server in March of 2023. If the protein had close hits to viruses but no eukaryote genomes, we decided that it could be a contaminant virus and not endogenized. If the protein had hits to proteins from other eukaryotic genomes but was consistently annotated to encode a non-capsid protein, we considered it to be a false positive. Some predicted proteins repeatedly came up as false-positive hits, which likely reflects a shared domain with our viral HMMs. Common hits that fell into this category were Clp protease, ovarian tumor domain (OTU), LRR domain, IG domain, hemicentin, obscurin, peroxidasin, titin, twitchin, fasciclin, neural cell adhesion molecule, roundabout, AAA ATPase, and 26S proteasome subunit. After manual curation, we were left with 14 putative endogenized capsids (not counting our expected, positive-control hit — the vp39 HMM to the ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Microplitis demolitor",
				},
				{
					type: "text",
					text: " major bracovirus capsid protein vp39). The highest-confidence endogenized capsids we recovered were from four parasitoid wasp species and one spider.",
				},
				{
					type: "hard_break",
					attrs: {
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
			],
		},
		{
			type: "blockquote",
			attrs: {
				id: "n382bsbwgaq",
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "paragraph",
					attrs: {
						id: "nikssfcf7xz",
						class: null,
						textAlign: null,
						rtl: null,
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "text",
							text: "Download our ",
						},
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
							],
							text: "manually annotated results",
						},
						{
							type: "text",
							text: " from searching VOG HMMs against venom transcriptomes on ",
						},
						{
							type: "text",
							marks: [
								{
									type: "link",
									attrs: {
										href: "https://zenodo.org/records/12775362/files/Annotated_capsid_no_cutoffs_summary_with_seqs_05092024.xlsx?download=1",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "Zenodo",
						},
						{
							type: "text",
							text: ".",
						},
					],
				},
			],
		},
		{
			type: "heading",
			attrs: {
				level: 2,
				fixedId: "",
				id: "horizontal-gene-transfer-hgt-analysis-of-venomous-animal-genomes",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Horizontal gene transfer (HGT) analysis of venomous animal genomes ",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nkrxfjdztwe",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Because the evolutionary acquisition of viral genes is a key example of inter-kingdom horizontal gene transfer (HGT), we decided to also deploy an approach that was specifically tailored to detect inter-kingdom HGT events. We used our preHGT ",
				},
				{
					type: "citation",
					attrs: {
						id: "n9ccze633a9",
						href: null,
						value: "https://doi.org/10.57844/ARCADIA-JFBP-7P11",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: " pipeline to scan genus-level pangenomes for recent viral HGT events using a BLAST-based taxonomic approach. We used gene models for nine venomous species: ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ixodes scapularis, Ixodes persulcatus, Microplitis demolitor, Sepia pharaonis, Trichomalopsis sarcophagae, Ampulex compressa, Ophiophagus hannah, Bothrops jararaca, ",
				},
				{
					type: "text",
					text: "and ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Naja naja",
				},
				{
					type: "text",
					text: ".",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "n9kgncywlc4",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "We created nucleotide pangenomes at the genus level by clustering genes at 90% length and sequence identity to determine the unique set of genes for the genus. We then chose a representative sequence for each cluster by selecting the sequence with the most alignments. We used BLASTp to compare representative proteins to a clustered non-redundant (nr) database, inspired by NCBI’s ",
				},
				{
					type: "text",
					marks: [
						{
							type: "link",
							attrs: {
								href: "https://ncbiinsights.ncbi.nlm.nih.gov/2022/05/02/clusterednr_1/",
								title: null,
								target: "_blank",
								pubEdgeId: null,
							},
						},
					],
					text: "ClusteredNR",
				},
				{
					type: "text",
					text: ". The clustered database ",
				},
				{
					type: "citation",
					attrs: {
						id: "n63jrfhg65u",
						href: null,
						value: "https://doi.org/10.57844/ARCADIA-W8XT-PC81",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: " allowed us to capture more taxonomically diverse hits.",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "n1gy0xre13z",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "We then used the “alien index” to predict whether an HGT event occurred ",
				},
				{
					type: "citation",
					attrs: {
						id: "n42539lubtz",
						href: null,
						value: "https://doi.org/10.1126/science.1156407",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ". The alien index is a metric that tells us how similar a given gene is to genes in related, non-self “acceptor” species, compared to unrelated potential “donor” species. In the case of horizontal gene transfer, we should see genes that have high similarity to genes in the donor species and low similarity to genes in species closely related to the acceptor species. We calculate the alien index by subtracting the E-value of the best donor (viral) hit from the E-value of the best non-self acceptor hit. If the best viral hit E-value is closer to zero than the best venomous species hit, the alien index will be positive. We used the following previously published thresholds ",
				},
				{
					type: "citation",
					attrs: {
						id: "nhkughsptrg",
						href: null,
						value: "https://doi.org/10.3390/genes8100248",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: " for determining HGT events: an alien index > 0 is possible HGT, > 15 is likely HGT, and > 45 is highly likely. We removed potential contaminants from this candidate list that we only saw in one genome within an investigated genus and that either had very high (> 90%) identity to a viral sequence or that were very short with high identity (< 20,000 bp, > 70% identity). We performed ortholog annotation (KEGG, PFAM, and viral and biosynthetic gene clusters) for proteins that may have been horizontally transferred.",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "n1cpuamd4ab",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "The pipeline produced a table of possible HGT events, including the predicted donor and acceptor taxa, alien index and BLASTp values, ortholog annotations, and genomic location information for this protein in the “acceptor” genome. Each hit is scored based on its likelihood to be either a real HGT event vs. contamination. We filtered out all events that were scored as “likely contamination.” After filtering, we detected 66 putative HGT events, 21 of which were inter-kingdom events predicted to be from a viral donor. ",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "n3y2gaamnry",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "For the six putative viral HGT-derived genes in ticks (",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ixodes persulcatus",
				},
				{
					type: "text",
					text: " and ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ixodes scapularis",
				},
				{
					type: "text",
					text: "), we used tBLASTn to see if similar genes were also integrated into the ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Amblyomma americanum",
				},
				{
					type: "text",
					text: " genome, which we hadn’t included in this HGT analysis since we didn't have gene predictions for this species at the time of analysis. We observed multiple hits for KAG0427517.1 and KAG0420414.1 in the ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "A. americanum ",
				},
				{
					type: "text",
					text: "genome, suggesting that such viral HGT events may be common in ticks.",
				},
				{
					type: "hard_break",
					attrs: {
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
			],
		},
		{
			type: "blockquote",
			attrs: {
				id: "nmfl5xgu0w7",
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "paragraph",
					attrs: {
						id: "nqyz3q5rgi2",
						class: null,
						textAlign: null,
						rtl: null,
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "text",
							text: "You can find our ",
						},
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
								{
									type: "link",
									attrs: {
										href: "https://zenodo.org/records/12775362/files/Annotated_hgt_results_venom_050924.xlsx?download=1",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "manually annotated results",
						},
						{
							type: "text",
							text: " from running preHGT on the genomes of venomous species here and the ",
						},
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
								{
									type: "link",
									attrs: {
										href: "https://zenodo.org/records/12775362/files/preHGT_input_genomes.csv?download=1",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "genomes we input into preHGT",
						},
						{
							type: "text",
							text: ", both on ",
						},
						{
							type: "text",
							marks: [
								{
									type: "link",
									attrs: {
										href: "https://zenodo.org/records/12775362",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "Zenodo",
						},
						{
							type: "text",
							text: ".",
						},
					],
				},
			],
		},
		{
			type: "blockquote",
			attrs: {
				id: "n1x4gcoog4v",
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "paragraph",
					attrs: {
						id: "nq4xzktvolg",
						class: null,
						textAlign: null,
						rtl: null,
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "text",
							text: "You can find the ",
						},
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
							],
							text: "preHGT GitHub repo",
						},
						{
							type: "text",
							text: " ",
						},
						{
							type: "text",
							marks: [
								{
									type: "link",
									attrs: {
										href: "https://github.com/Arcadia-Science/prehgt",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "here",
						},
						{
							type: "text",
							text: ".",
						},
					],
				},
			],
		},
		{
			type: "heading",
			attrs: {
				level: 2,
				fixedId: "",
				id: "genomic-context-analysis-of-putative-endogenized-viral-genes",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Genomic context analysis of putative endogenized viral genes ",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nqs9kz6jbe3",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "To evaluate if viral genes are integrated into the eukaryotic genome (rather than a viral contaminant), we pulled the contigs for each hit from the DBSOURCE field on the protein’s GenBank page. We displayed these contigs using Geneious Prime (version 2022.2.1) and manually analyzed the contigs to assess if they're part of the tick genome by looking for introns, a eukaryote-specific feature, in the capsid gene and predicting origin and function of neighboring genes using the BLAST, HHpred, and Foldseek web servers.",
				},
			],
		},
		{
			type: "blockquote",
			attrs: {
				id: "nmslu3ry40d",
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "paragraph",
					attrs: {
						id: "nvqhpdtlrpi",
						class: null,
						textAlign: null,
						rtl: null,
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "text",
							text: "You can find ",
						},
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
							],
							text: "GenBank files containing the genomic neighborhood ",
						},
						{
							type: "text",
							text: "of putatively horizontally transferred viral genes in the tick genome ",
						},
						{
							type: "text",
							marks: [
								{
									type: "link",
									attrs: {
										href: "https://zenodo.org/records/12775362",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "here",
						},
						{
							type: "text",
							text: ". We’ve shared seven GenBank files on Zenodo — one for each contig, named with the putative viral protein identifier.",
						},
					],
				},
			],
		},
		{
			type: "heading",
			attrs: {
				level: 2,
				fixedId: "",
				id: "salivary-expression-analysis-of-putative-endogenized-viral-genes",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Salivary expression analysis of putative endogenized viral genes",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nbfrodnx3je",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "To evaluate whether our viral proteins of interest are expressed in tick saliva, we BLASTed our top seven candidate proteins against ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ixodes scapularis and Ixodes ricinus ",
				},
				{
					type: "text",
					text: "proteins",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: " ",
				},
				{
					type: "text",
					text: "from the NCBI TSA (accession numbers: GADI01P.1, GEGO01P.1, GHXN01.1, GIFC01P.1, and GKHW01.1) using Geneious Prime (version 2022.2.1).",
				},
			],
		},
		{
			type: "blockquote",
			attrs: {
				id: "nl69qohazd9",
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "paragraph",
					attrs: {
						id: "nbvi9vbqzat",
						class: null,
						textAlign: null,
						rtl: null,
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "text",
							text: "Download our ",
						},
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
							],
							text: "BLAST results",
						},
						{
							type: "text",
							text: " from searching putatively endogenized viral proteins against tick salivary transcriptomes on ",
						},
						{
							type: "text",
							marks: [
								{
									type: "link",
									attrs: {
										href: "https://zenodo.org/records/12775362/files/TSA_blast_results.csv?download=1",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "Zenodo",
						},
						{
							type: "text",
							text: ".",
						},
					],
				},
			],
		},
		{
			type: "heading",
			attrs: {
				level: 2,
				fixedId: "",
				id: "foldseek-analysis-of-putative-endogenized-viral-genes",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Foldseek analysis of putative endogenized viral genes",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nkd759oktas",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "To search for structural homologs of our capsid proteins of interest in other species, we used Foldseek, a tool that enables structure-based searches for similar proteins ",
				},
				{
					type: "citation",
					attrs: {
						id: "necgrihm8fa",
						href: null,
						value: "https://doi.org/10.1038/s41587-023-01773-0",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ". We prioritized three of the hits from the HMM- and HGT-based search (",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ixodes",
				},
				{
					type: "text",
					text: " rhabdovirus nucleocapsid KAG0427517, ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ixodes",
				},
				{
					type: "text",
					text: " sabavirus nucleocapsid EEC17452, and ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ixodes",
				},
				{
					type: "text",
					text: " Gag-Pol XP_042148722). We also queried with the wasp bracovirus capsid NP_001401748 that originally inspired this project. There was a structure available in the AlphaFold database for the ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ixodes",
				},
				{
					type: "text",
					text: " sabavirus nucleocapsid (AF-B7QF30), but not for the other three proteins. We folded the other three using ColabFold (version 1.5.2) ",
				},
				{
					type: "citation",
					attrs: {
						id: "nqslnlzl7sk",
						href: null,
						value: "https://doi.org/10.1038/s41592-022-01488-1",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: " and used these to search for structures with the Foldseek web tool (version Foldseek 5-53465f0). ",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nbjwocstcf8",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "We searched all three AlphaFold databases available on Foldseek (i.e., AlphaFold/UniProt50 version 4, AlphaFold/Swiss-Prot version 4, and AlphaFold/Proteome version 4) ",
				},
				{
					type: "citation",
					attrs: {
						id: "npygyt771iu",
						href: null,
						value: "https://doi.org/10.1038/s41592-022-01488-1",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "citation",
					attrs: {
						id: "nq3kj45v991",
						href: null,
						value: "https://doi.org/10.1093/nar/gkab1061",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ". One important note is that the AlphaFold database specifically excluded any viral proteins, so any protein hits are likely not coming from the viruses themselves. We then filtered these hits using Foldseek E-values (similar to BLAST E-values), to determine the confidence of their hits. The developers of Foldseek consider E-values less than 0.01 to represent homologous pairs ",
				},
				{
					type: "citation",
					attrs: {
						id: "nmb9dnswga9",
						href: null,
						value: "https://doi.org/10.1038/s41586-023-06510-w",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ", so we applied that cutoff here as well for the bracovirus capsid, the ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ixodes",
				},
				{
					type: "text",
					text: " sabavirus nucleocapsid, and the ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ixodes",
				},
				{
					type: "text",
					text: " rhabdovirus nucleocapsid, which each returned a low number of hits (< 20). However, the ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ixodes",
				},
				{
					type: "text",
					text: " Gag-Pol protein had a very wide distribution, so we used a more stringent E-value cutoff of 10",
				},
				{
					type: "text",
					marks: [
						{
							type: "sup",
						},
					],
					text: "−3",
				},
				{
					type: "text",
					text: " to narrow in on top hits, identifying 924 structural homologs.",
				},
			],
		},
		{
			type: "blockquote",
			attrs: {
				id: "nnr7lo9cqxl",
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "paragraph",
					attrs: {
						id: "nxlxawcnde2",
						class: null,
						textAlign: null,
						rtl: null,
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "text",
							text: "Download a ",
						},
						{
							type: "text",
							marks: [
								{
									type: "link",
									attrs: {
										href: "https://zenodo.org/records/12775362/files/Foldseek_capsid_followup_05092024.xlsx?download=1",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "CSV",
						},
						{
							type: "text",
							text: " containing ",
						},
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
							],
							text: "results from searching putatively endogenized viral capsid proteins against AlphaFold",
						},
						{
							type: "text",
							text: "/UniProt50 (version 4), AlphaFold/Swiss-Prot (version 4), and AlphaFold/Proteome (version 4) using the Foldseek web server.",
						},
					],
				},
			],
		},
		{
			type: "heading",
			attrs: {
				level: 2,
				fixedId: "",
				id: "structural-clustering-to-identify-capsids-in-ornithodoros-turicata",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Structural clustering to identify capsids in ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ornithodoros turicata",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "n3eyr8hnf52",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "We tested a different approach to identify capsids that have been endogenized by ticks using structure-based clustering and the ProteinCartography pipeline ",
				},
				{
					type: "citation",
					attrs: {
						id: "nse7oxmjhwh",
						href: null,
						value: "https://doi.org/10.57844/arcadia-a5a6-1068",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ". We co-clustered viral proteins with tick proteins and identified clusters where tick proteins co-cluster with known viral capsids.",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nvvhgctdio1",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "For this analysis, we first needed to obtain virus capsid protein structures for comparison. As stated previously, the AlphaFold database doesn't currently include viral proteins. Therefore, we performed structural predictions using ESMFold for all capsid-related proteins from the VOG database. We chose to use ESMFold in this case because it’s faster than AlphaFold, which was important for the number of proteins we needed to fold ",
				},
				{
					type: "citation",
					attrs: {
						id: "nn9efphrjyq",
						href: null,
						value: "https://doi.org/10.1126/science.ade2574",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ".",
				},
			],
		},
		{
			type: "blockquote",
			attrs: {
				id: "nhda8y7pwe0",
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "paragraph",
					attrs: {
						id: "nd8ex9mpv4c",
						class: null,
						textAlign: null,
						rtl: null,
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "text",
							text: "These ",
						},
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
							],
							text: "predicted capsid protein structures",
						},
						{
							type: "text",
							text: " are on ",
						},
						{
							type: "text",
							marks: [
								{
									type: "link",
									attrs: {
										href: "https://zenodo.org/records/12796464",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "Zenodo",
						},
						{
							type: "text",
							text: " in the “",
						},
						{
							type: "text",
							marks: [
								{
									type: "link",
									attrs: {
										href: "https://zenodo.org/records/12796464/files/structures.zip?download=1",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "structures.zip",
						},
						{
							type: "text",
							text: "” file.",
						},
					],
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "n8uc28o4mgm",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "We searched UniProt for all proteins from ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ornithodoros turicata ",
				},
				{
					type: "text",
					text: "(UniProt organism ID: 34597)",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: ". ",
				},
				{
					type: "text",
					text: "We chose to start with this particular tick because there are a tractable number (< 10,000) of structures from ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "O. turicata ",
				},
				{
					type: "text",
					text: "on UniProt, and many of these structures were predicted from its",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: " ",
				},
				{
					type: "text",
					text: "salivary transcriptome. We then downloaded the associated metadata and structures using scripts from the ProteinCartography repo ",
				},
				{
					type: "citation",
					attrs: {
						id: "nurxhl9nsam",
						href: null,
						value: "https://doi.org/10.57844/arcadia-a5a6-1068",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ".",
				},
			],
		},
		{
			type: "blockquote",
			attrs: {
				id: "nyzog6j6l8a",
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "paragraph",
					attrs: {
						id: "ngos3gi9js2",
						class: null,
						textAlign: null,
						rtl: null,
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
							],
							text: "Python scripts",
						},
						{
							type: "text",
							text: " for downloading metadata are ",
						},
						{
							type: "text",
							marks: [
								{
									type: "link",
									attrs: {
										href: "https://github.com/Arcadia-Science/ProteinCartography/blob/v0.4.2/ProteinCartography/fetch_uniprot_metadata.py",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "here",
						},
						{
							type: "text",
							text: ", the ",
						},
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
							],
							text: "script",
						},
						{
							type: "text",
							text: " for downloading structures is ",
						},
						{
							type: "text",
							marks: [
								{
									type: "link",
									attrs: {
										href: "https://github.com/Arcadia-Science/ProteinCartography/blob/v0.4.2/ProteinCartography/download_pdbs.py",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "here",
						},
						{
							type: "text",
							text: ", and ",
						},
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
							],
							text: "notebooks",
						},
						{
							type: "text",
							text: " to facilitate metadata and structure downloads are in ",
						},
						{
							type: "text",
							marks: [
								{
									type: "link",
									attrs: {
										href: "https://github.com/Arcadia-Science/capsids/tree/pubv1.0",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "this GitHub repo",
						},
						{
							type: "text",
							text: ".",
						},
					],
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "n7fflv8pw07",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "To test this co-clustering approach, we performed ProteinCartography clustering analyses on combinations of viral structures and tick structures. This run contained the entire collection of capsid-related VOG proteins and all of the tick proteins from ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "O. turicata",
				},
				{
					type: "text",
					text: ". ProteinCartography output files include interactive UMAP and t-SNE embeddings with metadata overlays. Also included in ProteinCartography’s outputs are the corresponding TSV files that contain plotting information and all the metadata for each analysis, an interactive heatmap showing the similarity within and between clusters, and all the intermediate outputs. We looked for structural clusters in which viral proteins co-clustered with tick proteins. We found that the ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ornithodoros ",
				},
				{
					type: "text",
					text: "proteins A0A2R5L5R2 and A0A2R5L7H1 cluster with proteins from VOG00029, VOG05312, VOG06972, and VOG20608. ",
				},
			],
		},
		{
			type: "blockquote",
			attrs: {
				id: "nif85i15ci2",
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "paragraph",
					attrs: {
						id: "ntnoa9d8xbw",
						class: null,
						textAlign: null,
						rtl: null,
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "text",
							text: "The ",
						},
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
							],
							text: "data",
						},
						{
							type: "text",
							text: " we used for this analysis are on ",
						},
						{
							type: "text",
							marks: [
								{
									type: "link",
									attrs: {
										href: "https://zenodo.org/records/12796464",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "Zenodo",
						},
						{
							type: "text",
							text: ", the ",
						},
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
							],
							text: "code",
						},
						{
							type: "text",
							text: " is in ",
						},
						{
							type: "text",
							marks: [
								{
									type: "link",
									attrs: {
										href: "https://github.com/Arcadia-Science/ProteinCartography/tree/v0.4.2",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "this GitHub repo",
						},
						{
							type: "text",
							text: ", and ",
						},
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
							],
							text: "additional notebooks",
						},
						{
							type: "text",
							text: " to facilitate metadata and structure downloading and custom plotting are in ",
						},
						{
							type: "text",
							marks: [
								{
									type: "link",
									attrs: {
										href: "https://github.com/Arcadia-Science/capsids",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "this GitHub repo",
						},
						{
							type: "text",
							text: ".",
						},
					],
				},
			],
		},
		{
			type: "heading",
			attrs: {
				level: 2,
				fixedId: "",
				id: "taxonomic-tree-visualizations",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Taxonomic tree visualizations ",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nf0hhesxz82",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "We used the scientific species names from each analysis to create a taxonomic tree using phyloT (version 2) and the phyloT database (version 2023.2). Using the ",
				},
				{
					type: "text",
					marks: [
						{
							type: "link",
							attrs: {
								href: "https://phylot.biobyte.de/index.cgi",
								title: null,
								target: "_blank",
								pubEdgeId: null,
							},
						},
					],
					text: "phyloT web server",
				},
				{
					type: "text",
					text: ", we set the node identifiers to scientific names, used expanded internal nodes, set polytomy to yes, and exported the trees as Newick files that we visualized and further annotated in iTOL (version 6.9) ",
				},
				{
					type: "citation",
					attrs: {
						id: "nt92ou9mx66",
						href: null,
						value: "https://doi.org/10.1093/nar/gkab301",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ". ",
				},
			],
		},
		{
			type: "blockquote",
			attrs: {
				id: "n1k75ue38ff",
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "paragraph",
					attrs: {
						id: "nvs4zvadynb",
						class: null,
						textAlign: null,
						rtl: null,
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
							],
							text: "Newick files for the trees",
						},
						{
							type: "text",
							text: " in ",
						},
						{
							type: "reference",
							attrs: {
								id: "nmyogs2ib69",
								targetId: "neezumhdcet",
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
						},
						{
							type: "text",
							text: " and ",
						},
						{
							type: "reference",
							attrs: {
								id: "nafgt6i9cd6",
								targetId: "n41kwh0e03y",
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
						},
						{
							type: "text",
							text: " are available on ",
						},
						{
							type: "text",
							marks: [
								{
									type: "link",
									attrs: {
										href: "https://zenodo.org/records/12775362",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "Zenodo",
						},
						{
							type: "text",
							text: ".",
						},
					],
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "ndxh3attl17",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
		},
		{
			type: "blockquote",
			attrs: {
				id: "nylyp0om911",
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "paragraph",
					attrs: {
						id: "nvmlfrjpo4q",
						class: null,
						textAlign: null,
						rtl: null,
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "text",
							text: "All ",
						},
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
							],
							text: "code",
						},
						{
							type: "text",
							text: " generated and used for the pub is available in this GitHub repository (DOI: ",
						},
						{
							type: "text",
							marks: [
								{
									type: "link",
									attrs: {
										href: "https://doi.org/10.5281/zenodo.12809305",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "10.5281/zenodo.12809305",
						},
						{
							type: "text",
							text: "), including scripts for performing HMM and BLAST searches, plus notebooks for ProteinCartography analyses.",
						},
					],
				},
			],
		},
		{
			type: "heading",
			attrs: {
				level: 1,
				fixedId: "",
				id: "the-results",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "The results",
				},
			],
		},
		{
			type: "blockquote",
			attrs: {
				id: "nzu7la48srk",
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "paragraph",
					attrs: {
						id: "nb9k1ub5poc",
						class: null,
						textAlign: null,
						rtl: null,
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
							],
							text: "SHOW ME THE DATA",
						},
						{
							type: "text",
							text: ": Access data from our ",
						},
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
							],
							text: "capsid HMM search",
						},
						{
							type: "text",
							text: ", our ",
						},
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
							],
							text: "preHGT run",
						},
						{
							type: "text",
							text: " to find putatively endogenized viral capsids, and our ",
						},
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
							],
							text: "follow-up work",
						},
						{
							type: "text",
							text: " looking into individual hits with BLAST searches of tick salivary transcriptome, genomic neighborhood analysis, and Foldseek analysis on ",
						},
						{
							type: "text",
							marks: [
								{
									type: "link",
									attrs: {
										href: "https://zenodo.org/records/12775362",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "Zenodo",
						},
						{
							type: "text",
							text: " (DOI: ",
						},
						{
							type: "text",
							marks: [
								{
									type: "link",
									attrs: {
										href: "https://doi.org/10.5281/zenodo.12775362",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "10.5281/zenodo.12775362",
						},
						{
							type: "text",
							text: ").",
						},
					],
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "ntmsmcrxhh5",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
		},
		{
			type: "blockquote",
			attrs: {
				id: "noyli7r1c42",
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "paragraph",
					attrs: {
						id: "nn18t2ee9gh",
						class: null,
						textAlign: null,
						rtl: null,
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
							],
							text: "Data",
						},
						{
							type: "text",
							text: " from our ProteinCartography runs, including structures of all tick and viral capsid proteins, the configuration file, and all outputs, are in a separate ",
						},
						{
							type: "text",
							marks: [
								{
									type: "link",
									attrs: {
										href: "https://zenodo.org/records/12796464",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "Zenodo",
						},
						{
							type: "text",
							text: " record (DOI: ",
						},
						{
							type: "text",
							marks: [
								{
									type: "link",
									attrs: {
										href: "https://doi.org/10.5281/zenodo.12796464",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "10.5281/zenodo.12796464",
						},
						{
							type: "text",
							text: ").",
						},
					],
				},
			],
		},
		{
			type: "heading",
			attrs: {
				level: 2,
				fixedId: "",
				id: "sequence-based-searches-highlight-retrotransposon-capsid-like-genes-in-parasitoid-wasps-and-other-parasitic-species-including-ticks",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Sequence-based searches highlight retrotransposon capsid-like genes in parasitoid wasps and other parasitic species, including ticks",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nev108k8217",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "We first used a sequence-similarity approach to search for capsid-like sequences in venomous species. Because there's a large body of public data collected from venom samples, we were able to curate a database of 156 venom transcriptomes from 135 different species ",
				},
				{
					type: "citation",
					attrs: {
						id: "nfpdu1sa3u9",
						href: null,
						value: "https://doi.org/10.57844/arcadia-d3dc-7499",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ". We searched this database using viral capsid-specific hidden Markov model (HMM) profiles from the VOG database. Because the bracovirus capsid isn't represented in the VOG database, we also generated a custom bracovirus capsid HMM to use in searching against the venom transcriptomes.",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "n2jgc6tbah0",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Searches using VOG capsid HMMs generated over 3,000 hits before filtering. However, the bracovirus vp39 capsid HMM only hit the ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Microplitis demolitor",
				},
				{
					type: "text",
					text: " transcriptome and didn’t have any other high-confidence hits. We conclude that bracovirus-like capsids are not broadly present across animal venoms. That said, this transcriptome-based search likely underrepresents the true distribution of domesticated bracovirus capsids within parasitoid wasps. Previous work on bracoviruses shows that they’re primarily present in the calyx fluid from wasp ovaries, which is injected during oviposition along with venom gland secretions. While the ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Microplitus demolitor",
				},
				{
					type: "text",
					text: " transcriptome we used came from both the wasp venom gland and ovaries, the other parasitoid data we used only included transcripts from venom glands ",
				},
				{
					type: "citation",
					attrs: {
						id: "nvfxy2strze",
						href: null,
						value: "https://doi.org/10.1038/s42003-020-01623-8",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ". The lack of ovary transcriptome data may, in part, explain the absence of bracovirus sequences from those parasitoid wasp transcriptomes.",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "ns478ydyosi",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "We performed manual curation of the top 210 VOG HMM hits via BLAST, which suggested that many of the hits were false positives. Some false positives came from potential contaminating or venom virome-associated sequences, including bacteriophage capsid proteins. Other false positives came from hits of our viral HMMs to shared domains in eukaryotic genes that are probably not derived from recent viral capture and endogenization. Many of our false positive hits had immunoglobulin domains or protease domains, which are common to both viral proteins and host proteins. ",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nsxjv70kcjk",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "These filters limited our homologs of interest to 14 total hits (not including the vp39 bracovirus capsid hit in ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Microplitis demolitor) ",
				},
				{
					type: "text",
					text: "(",
				},
				{
					type: "reference",
					attrs: {
						id: "nfx04eesobq",
						targetId: "nheciqbgp4q",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: "). These viral hits came from four species of parasitoid wasp (",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Cotesia vestalis",
				},
				{
					type: "text",
					text: ", ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Diadromus collaris",
				},
				{
					type: "text",
					text: ", ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Microplitis demolitor",
				},
				{
					type: "text",
					text: ", and ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Pteromalus puparum",
				},
				{
					type: "text",
					text: "), and one spider (",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Phoneutria nigriventer",
				},
				{
					type: "text",
					text: "). We were initially disappointed in the results from this search because it mostly returned endogenized proteins in wasps, which are already known to have domesticated capsids. Our goal had been to identify endogenized capsid proteins outside of parasitoids, ideally in species of venomous animals that bite humans or other mammals. However, when examining the taxonomic distribution of the 14 hits using BLAST, we found that one of these proteins — a Gag-Pol protein from LTR-type retrotransposons — had homologs outside of parasitoid wasps, including in ticks, which bite and feed on humans.",
				},
			],
		},
		{
			type: "image",
			attrs: {
				id: "nheciqbgp4q",
				url: "https://assets.pubpub.org/14a1n879/Fig2_Venomous-sequence-based-01721329919003.png",
				size: 85,
				align: "center",
				caption:
					'<p id="nza6ycxm2c2" data-reactroot=""><strong>A sequence-based search of 135 species’ transcriptomes identifies putatively endogenized viral capsids expressed in venom</strong>.</p><p id="r2939193228" data-reactroot="">We searched venom transcriptomes from 135 species, which included venomous snakes, an annelid, mollusks, arachnids, centipedes, a crustacean, and parasitoid wasps. These species are displayed in a taxonomic tree. We did not find evidence of endogenized capsids in the venoms of most animals, indicated with black circles on leaf tips. However, four species of parasitoid wasp (<em>Cotesia vestalis</em>, <em>Diadromus collaris</em>, <em>Microplitis demolitor,</em> and<em> Pteromalus puparum</em>) have putatively endogenized capsids in their venoms, as does the Brazilian wandering spider (<em>Phoneutria nigriventer</em>). These instances of potential domestication are highlighted with red circles. Individual species names are omitted from this figure due to the size and complexity of the tree, but the associated Newick file can be found <a href="https://zenodo.org/records/12775362/files/Fig2_Newick_HMM_venoms.txt?download=1" target="_blank">here</a>, the list of input transcriptomes is <a href="https://github.com/Arcadia-Science/capsids/blob/main/metadata/SRA_TSA_venom_gland_accessions.csv" target="_blank">here</a>, and the hits from this sequence-based search are <a href="https://zenodo.org/records/12775362/files/Annotated_capsid_no_cutoffs_summary_with_seqs_05092024.xlsx?download=1" target="_blank">here</a>.</p>',
				altText: "",
				hideLabel: false,
				fullResolution: false,
				href: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
		},
		{
			type: "paragraph",
			attrs: {
				id: "nx0rsasguuw",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Interestingly, retrotransposon-derived Gag proteins form capsid-like structures, also known as viral-like particles (VLPs) ",
				},
				{
					type: "citation",
					attrs: {
						id: "n21s1zdw4e9",
						href: null,
						value: "https://doi.org/10.1186/gb-2004-5-6-225",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ". Further, Gag-Pol homologs from humans (Peg10) and ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Drosophila",
				},
				{
					type: "text",
					text: " (Arc) were recently shown to be involved in RNA delivery between cells ",
				},
				{
					type: "citation",
					attrs: {
						id: "nr4qchkzcbv",
						href: null,
						value: "https://doi.org/10.1016/j.cell.2017.12.024",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "citation",
					attrs: {
						id: "n423klldoz7",
						href: null,
						value: "https://doi.org/10.1126/science.abg6155",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ". We found that our wasp Gag-Pol protein of interest had homologs in parasitic species, including aphids, mosquitoes, and ticks. Of these species, we were particularly interested in the presence of capsid-like proteins in ticks, given our previous work to develop resources for studying the host-manipulating properties of tick saliva ",
				},
				{
					type: "citation",
					attrs: {
						id: "n6omh0jbdso",
						href: null,
						value: "https://doi.org/10.57844/arcadia-3hyh-3h83",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ". Given that the salivary glands of ticks secrete venom protein homologs into their host upon biting, ticks can be considered venomous species ",
				},
				{
					type: "citation",
					attrs: {
						id: "nsc64ntpe52",
						href: null,
						value: "https://doi.org/10.1186/1742-9994-11-47",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ". For these reasons, we prioritized one of the tick Gag protein homologs for further analysis as part of our investigation into capsid-like proteins in venoms (XP_042148722.1 from ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ixodes scapularis",
				},
				{
					type: "text",
					text: ") (Table 1).",
				},
			],
		},
		{
			type: "heading",
			attrs: {
				level: 2,
				fixedId: "",
				id: "evidence-of-capsid-genes-horizontally-transferred-from-viruses-to-ixodes-tick-species",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Evidence of capsid genes horizontally transferred from viruses to ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ixodes",
				},
				{
					type: "text",
					text: " tick species",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nfwsry7s12l",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Because the evolutionary acquisition of viral genes such as capsids is an example of inter-kingdom horizontal gene transfer (HGT), we next deployed a software pipeline specifically tailored to detect inter-kingdom HGT events — preHGT ",
				},
				{
					type: "citation",
					attrs: {
						id: "npibqhf7g9p",
						href: null,
						value: "https://doi.org/10.57844/arcadia-jfbp-7p11",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ". Of the original venomous species included in our list, only nine had gene models available through NCBI (a prerequisite for preHGT) at the time we ran the pipeline. Although the original venomous species dataset we used for HMM-based analyses didn’t include tick species, we decided to include the ticks ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ixodes scapularis",
				},
				{
					type: "text",
					text: " and ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ixodes persulcatus",
				},
				{
					type: "text",
					text: " in our HGT analysis because we’d found a capsid protein in ticks in our follow-up BLAST searches, described above.",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "njawn8k5vje",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "The preHGT analysis revealed 66 potential horizontal transfer events between our target species and all available genomes on NCBI. Of those, there were 21 instances of putative gene transfer from viruses to our species of interest. 14 of these were to parasitoid wasps, 11 of which were the previously studied bracovirus found within the ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Microplitisdemolitor ",
				},
				{
					type: "text",
					text: "genome. We identified seven viral HGT events outside of wasps, six of which we found in ticks (Table 1). The other non-wasp putative viral HGT event was of a nucleoside transporter in ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Sepia pharaonis",
				},
				{
					type: "text",
					text: ", the pharaoh cuttlefish.",
				},
			],
		},
		{
			type: "image",
			attrs: {
				id: "neezumhdcet",
				url: "https://assets.pubpub.org/xcr9lti7/Fig3_Venomous-HGT-based-61721330313743.png",
				size: 100,
				align: "center",
				caption:
					'<p id="n0dyswhi6ao" data-reactroot=""><strong>An HGT-based search reveals putative viral HGT events in the genomes of venomous animals</strong>.</p><p id="r5380540706" data-reactroot="">We used the horizontal gene transfer (HGT) detection tool “preHGT” to identify putative viral HGT events in the genomes of nine venomous animals, including snakes, a cuttlefish, ticks, and parasitoid wasps. Evolutionary relationships between the species we searched are shown on a tree. We identified putative viral HGT events in ticks, parasitoid wasps, and the pharaoh cuttlefish. These events are marked by red circles at the leaf tips. The Newick file we used to generate this visual can be found <a href="https://zenodo.org/records/12775362/files/Fig3_Newick_HGT_genomes.txt?download=1" target="_blank">here</a>, the list of input genomes <a href="https://zenodo.org/records/12775362/files/preHGT_input_genomes.csv?download=1" target="_blank">here</a>, and the hits from this HGT search <a href="https://zenodo.org/records/12775362/files/Annotated_hgt_results_venom_050924.xlsx?download=1">here</a>.</p>',
				altText: "",
				hideLabel: false,
				fullResolution: true,
				href: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
		},
		{
			type: "paragraph",
			attrs: {
				id: "nko51phff0i",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Four of the six tick viral HGT events were predicted to be capsid-related sequences. These were all annotated as nucleocapsid proteins, or “N proteins,” from several different negative-stranded RNA viruses. These RNA viruses belong to groups known to infect ticks, such as Bunyavirales and Rhabdoviridae, yet these types of viruses don't go through a DNA phase in their replication. Integrated RNA viral genes have previously been noted in tick genomes, though their biological significance is unknown ",
				},
				{
					type: "citation",
					attrs: {
						id: "nghzon5btk9",
						href: null,
						value: "10.1101/2022.08.17.504163",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ". It’s intriguing to see RNA-derived viral nucleocapsids integrated into DNA-based tick genomes. The two other virus-to-tick HGT events in the dataset were both viral RNA-dependent RNA polymerase (RdRp). Nucleocapsid proteins from these RNA viruses are involved in binding and encapsulating RNA virus genomes and interact with viral RdRp to initiate viral genome replication ",
				},
				{
					type: "citation",
					attrs: {
						id: "nijybifkjkq",
						href: null,
						value: "https://doi.org/10.1002/wrna.1326",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "citation",
					attrs: {
						id: "nzr12hb21mm",
						href: null,
						value: "https://doi.org/10.3390/v12080835",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ". Given these interesting findings, we decided to prioritize all of these additional tick proteins for further analysis (Table 1).",
				},
			],
		},
		{
			type: "table",
			attrs: {
				id: "nvyypipzwot",
				hideLabel: true,
				align: null,
				size: null,
				smallerFont: false,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "table_row",
					attrs: {
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "table_header",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [87],
								background: "rgb(239, 239, 239)",
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "n9ylrxherq9",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "strong",
												},
												{
													type: "sup",
												},
											],
											text: "Protein",
										},
									],
								},
							],
						},
						{
							type: "table_header",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: "rgb(239, 239, 239)",
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "n194773cs5v",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "strong",
												},
												{
													type: "sup",
												},
											],
											text: "Tick species",
										},
									],
								},
							],
						},
						{
							type: "table_header",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [123],
								background: "rgb(239, 239, 239)",
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "ntpvx5669se",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "strong",
												},
												{
													type: "sup",
												},
											],
											text: "Closest viral hit",
										},
									],
								},
							],
						},
						{
							type: "table_header",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: "rgb(239, 239, 239)",
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nxambrzd5x8",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "strong",
												},
												{
													type: "sup",
												},
											],
											text: "Annotation",
										},
									],
								},
							],
						},
						{
							type: "table_header",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [163],
								background: "rgb(239, 239, 239)",
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "n6sq6ycdli7",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "strong",
												},
												{
													type: "sup",
												},
											],
											text: "Protein family",
										},
									],
								},
							],
						},
						{
							type: "table_header",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [111],
								background: "rgb(239, 239, 239)",
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nadgu0x23sc",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "strong",
												},
												{
													type: "sup",
												},
											],
											text: "Contains introns?",
										},
									],
								},
							],
						},
						{
							type: "table_header",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [107],
								background: "rgb(239, 239, 239)",
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "n1gtahokxmq",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "strong",
												},
												{
													type: "sup",
												},
											],
											text: "Expressed in salivary gland?",
										},
									],
								},
							],
						},
						{
							type: "table_header",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: "rgb(239, 239, 239)",
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "npbnbd5ksxv",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "strong",
												},
												{
													type: "sup",
												},
											],
											text: "Run through Foldseek?",
										},
									],
								},
							],
						},
					],
				},
				{
					type: "table_row",
					attrs: {
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [87],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nhrfr0zp7p9",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "XP_042148722.1",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nzo5faf34qx",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "em",
												},
												{
													type: "sup",
												},
											],
											text: "Ixodes scapularis",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [123],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nd2mzbkung4",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "N/A",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "n4p8itb4y43",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "Retro-transposon Gag protein",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [163],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nbyjofeb93l",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "pfam03732: Retrotrans_gag; pfam13650:Asp_protease_2; pfam00098; zf-CCHC ",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [111],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nsb4gyqm4lt",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "N",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [107],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nsl7jls3swp",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "Y",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nfmp6dbmaj1",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "Y",
										},
									],
								},
								{
									type: "paragraph",
									attrs: {
										id: "n6y44oris81",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "(",
										},
										{
											type: "reference",
											attrs: {
												id: "nh1qkicrncn",
												targetId: "n41kwh0e03y",
												suggestionId: null,
												suggestionTimestamp: null,
												suggestionUserId: null,
												suggestionDiscussionId: null,
												suggestionKind: null,
												suggestionOriginalAttrs: null,
											},
											marks: [
												{
													type: "sup",
												},
											],
										},
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: ", C)",
										},
									],
								},
							],
						},
					],
				},
				{
					type: "table_row",
					attrs: {
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [87],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nz7i94gkqcv",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "KAG0420414.1",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "n90j282kp5k",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "em",
												},
												{
													type: "sup",
												},
											],
											text: "Ixodes persulcatus",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [123],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "ncal30oeqar",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "Tahe rhabdovirus",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "ngbuy06ba7g",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "N protein",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [163],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nmpa8om0wj1",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "pfam00945: Rhabdo_ncap",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [111],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nz2mbq0msr0",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "Y",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [107],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nr6y8n5zg8b",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "Y",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "n43t69dp63i",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "N",
										},
									],
								},
							],
						},
					],
				},
				{
					type: "table_row",
					attrs: {
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [87],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nh92emy7o8q",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "KAG0427517.1",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nu7jwh8b83x",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "em",
												},
												{
													type: "sup",
												},
											],
											text: "Ixodes persulcatus",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [123],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nnqwbty40gh",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "Tahe rhabdovirus 3",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "n3do866spgg",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "N protein",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [163],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "npzhx3gamjh",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "pfam00945: Rhabdo_ncap",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [111],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "n31rqowvey2",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "Y",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [107],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nfc0lo4szam",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "Y",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nckn9n95pro",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "Y",
										},
									],
								},
								{
									type: "paragraph",
									attrs: {
										id: "naay5eoi6t4",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "(",
										},
										{
											type: "reference",
											attrs: {
												id: "ntog84y2kda",
												targetId: "n41kwh0e03y",
												suggestionId: null,
												suggestionTimestamp: null,
												suggestionUserId: null,
												suggestionDiscussionId: null,
												suggestionKind: null,
												suggestionOriginalAttrs: null,
											},
											marks: [
												{
													type: "sup",
												},
											],
										},
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: ", B)",
										},
									],
								},
							],
						},
					],
				},
				{
					type: "table_row",
					attrs: {
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [87],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nma28vvnqvs",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "EEC12039.1",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nl9eequ4u47",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "em",
												},
												{
													type: "sup",
												},
											],
											text: "Ixodes scapularis",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [123],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nk913pc9p1s",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "Sara tick phlebovirus",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nz5qs58ddz6",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "N protein",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [163],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nk19p43vpo2",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "pfam05733: Tenui_N",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [111],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "n5c1qs06brf",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "Y (gene is fragmented)",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [107],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "n7c4tsu1s1m",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "N",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nu28j251qbu",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "N",
										},
									],
								},
							],
						},
					],
				},
				{
					type: "table_row",
					attrs: {
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [87],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nepvvadswnx",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "EEC17452.1",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "noive95xep5",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "em",
												},
												{
													type: "sup",
												},
											],
											text: "Ixodes scapularis",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [123],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nsrj6ki35pb",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "South Bay sabavirus (bunyavirus order) ",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "n89p3kgag6l",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "N protein",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [163],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "n6s9dhzjpnx",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "pfam02477: Nairo_nucleo",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [111],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "n0tll2uqais",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "Y (gene is fragmented)",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [107],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nvkqftxgmyx",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "Y",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nmgwy5ey96b",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "Y",
										},
									],
								},
								{
									type: "paragraph",
									attrs: {
										id: "nircv4s9r7e",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "(",
										},
										{
											type: "reference",
											attrs: {
												id: "nc1h3qukn3b",
												targetId: "n41kwh0e03y",
												suggestionId: null,
												suggestionTimestamp: null,
												suggestionUserId: null,
												suggestionDiscussionId: null,
												suggestionKind: null,
												suggestionOriginalAttrs: null,
											},
											marks: [
												{
													type: "sup",
												},
											],
										},
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: ", A)",
										},
									],
								},
							],
						},
					],
				},
				{
					type: "table_row",
					attrs: {
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [87],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nk2vbmx779o",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "KAG0443635.1",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "n3gswdm0hm4",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "em",
												},
												{
													type: "sup",
												},
											],
											text: "Ixodes persulcatus",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [123],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nkjil42qcdw",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "Totiviridae sp.",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nrygl9qo75a",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "RdRp",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [163],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nwujxxut5ea",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "N/A",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [111],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "n6wgm5qa334",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "Y",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [107],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nvjoaj2sn7g",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "Y",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nn4jakh1n92",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "N",
										},
									],
								},
							],
						},
					],
				},
				{
					type: "table_row",
					attrs: {
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [87],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "ng1gsjhbh6g",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "KAG0444350.1",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "n32bj4w501w",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "em",
												},
												{
													type: "sup",
												},
											],
											text: "Ixodes persulcatus",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [123],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nwq4mnn06du",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "Phenuiviridae",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nn7nyif68mx",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "RdRp",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [163],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nure8ikbqdg",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "pfam12603: Bunya_L_PA-C-like",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [111],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nv4h1spcbmx",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "Y",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [107],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "n1qic73r6cc",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "N",
										},
									],
								},
							],
						},
						{
							type: "table_cell",
							attrs: {
								colspan: 1,
								rowspan: 1,
								colwidth: [109],
								background: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "paragraph",
									attrs: {
										id: "nmk34hwh8ps",
										class: null,
										textAlign: null,
										rtl: null,
										suggestionId: null,
										suggestionTimestamp: null,
										suggestionUserId: null,
										suggestionDiscussionId: null,
										suggestionKind: null,
										suggestionOriginalAttrs: null,
									},
									content: [
										{
											type: "text",
											marks: [
												{
													type: "sup",
												},
											],
											text: "N",
										},
									],
								},
							],
						},
					],
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "n4yajpnk4j2",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					marks: [
						{
							type: "strong",
						},
						{
							type: "sup",
						},
					],
					text: "Table 1",
				},
				{
					type: "text",
					marks: [
						{
							type: "sup",
						},
					],
					text: ".",
				},
				{
					type: "text",
					marks: [
						{
							type: "strong",
						},
						{
							type: "sup",
						},
					],
					text: " Tick proteins of potential viral origin",
				},
				{
					type: "text",
					marks: [
						{
							type: "sup",
						},
					],
					text: ". ",
				},
				{
					type: "hard_break",
					attrs: {
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					marks: [
						{
							type: "sup",
						},
					],
				},
				{
					type: "text",
					marks: [
						{
							type: "sup",
						},
					],
					text: "This table lists the seven tick proteins of potential viral origin that we found in this study. We identified the first protein on this list (XP_042148722.1) as a homolog of a viral capsid endogenized by a parasitoid wasp. We identified the other six proteins using HGT-based analysis of animal genomes. This table shows the protein accession number and associated metadata: the tick species in which we found the protein, the virus that the gene probably originated from, and the protein’s annotation and protein family. We’re also showing the results of our manual genomic context analysis of each gene, where we looked for signs of domestication and salivary expression. We assessed whether each gene is inserted into the tick genome, if it has introns, and if it’s expressed in the salivary gland. Last, we highlight the proteins for which we did follow-up analysis using Foldseek and note the associated figure numbers. Annotated GenBank files of the contigs for each protein are available ",
				},
				{
					type: "text",
					marks: [
						{
							type: "link",
							attrs: {
								href: "https://zenodo.org/records/12775362",
								title: null,
								target: "_blank",
								pubEdgeId: null,
							},
						},
						{
							type: "sup",
						},
					],
					text: "here",
				},
				{
					type: "text",
					marks: [
						{
							type: "sup",
						},
					],
					text: ".",
				},
			],
		},
		{
			type: "heading",
			attrs: {
				level: 2,
				fixedId: "",
				id: "capsid-genes-identified-in-ticks-contain-introns-and-are-expressed-in-salivary-glands",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Capsid genes identified in ticks contain introns and are expressed in salivary glands",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "n8kr3x2ee41",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "The HMM- and HGT-based searches led us to identify two different types of potential viral capsids in tick genomes — retrotransposon Gag-Pol capsid-like proteins and negative-stranded RNA viral nucleocapsids and associated RdRps. We next examined the genomic context of each of these genes to look for further evidence of endogenization. All seven genes are located within larger tick genomic contigs, suggesting integration into the tick genome rather than viral contamination that was sequenced along with the tick genome. Six out of seven contained introns (",
				},
				{
					type: "reference",
					attrs: {
						id: "n83id39vlnu",
						targetId: "nd2wtjuapb1",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: "), which suggests that the genes have been present in the tick genomes long enough to acquire eukaryotic genome characteristics. This result adds confidence that these sequences aren't derived from viral contaminants in the original datasets but, rather, are endogenized viral genes.",
				},
			],
		},
		{
			type: "image",
			attrs: {
				id: "nd2wtjuapb1",
				url: "https://assets.pubpub.org/kfizci85/Fig4_Tick-genomes-11721330518580.png",
				size: 100,
				align: "center",
				caption:
					'<p id="nzle2tyxaew" data-reactroot=""><strong>Genomic context of two viral capsid genes endogenized in tick genomes</strong>.</p><p id="r1992780397" data-reactroot="">Genomic context of the genes encoding KAG0427517.1, the rhabdovirus-derived N protein in <em>Ixodes persulcatus</em>, and XP_042148722.1, the Gag-Pol protein in <em>Ixodes scapularis</em>. Annotations (determined by looking for related proteins using BLAST, HHpred, and Foldseek) are included below each coding region. Genes of tick origin are grey-blue and viral-derived genes are dark orange. Introns are pale orange bars. Full annotated GenBank files of these contigs can be found <a href="https://zenodo.org/records/12775362/files/XP_042148722.1_contig.gb?download=1" target="_blank">here</a> and <a href="https://zenodo.org/records/12775362/files/KAG0427517.1_contig.gb?download=1" target="_blank">here</a>.</p>',
				altText: "",
				hideLabel: false,
				fullResolution: false,
				href: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
		},
		{
			type: "paragraph",
			attrs: {
				id: "nnploj4fw6x",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Because we found most of the viral capsid candidates using the HGT approach rather than the transcriptome analysis, we wondered whether these genes are expressed. We were particularly curious whether they’re expressed in tick saliva, as this would suggest that the genes are involved in venom-related functions. We found all of our candidate tick capsids in ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ixodes scapularis",
				},
				{
					type: "text",
					text: " and ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ixodes persulcatus",
				},
				{
					type: "text",
					text: "; however, there's no publicly available ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "I. persulcatus",
				},
				{
					type: "text",
					text: " salivary transcriptome. There are salivary transcriptomes from ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ixodes scapularis ",
				},
				{
					type: "text",
					text: "and",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: " Ixodes ricinus, ",
				},
				{
					type: "text",
					text: "so we used those to BLAST the capsid candidates. We found hits to five of the seven candidates, all in the ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "I. ricinus",
				},
				{
					type: "text",
					text: " salivary transcriptome, suggesting these genes have homologs in ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "I.",
				},
				{
					type: "text",
					text: " ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "ricinus",
				},
				{
					type: "text",
					text: " that are actively produced in the salivary glands.",
				},
			],
		},
		{
			type: "heading",
			attrs: {
				level: 2,
				fixedId: "",
				id: "searching-venoms-for-proteins-with-structures-similar-to-viral-capsids-identifies-capsid-like-proteins-in-multiple-blood-sucking-species-and-other-invertebrates",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Searching venoms for proteins with structures similar to viral capsids identifies capsid-like proteins in multiple blood-sucking species and other invertebrates",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nbbjf27l8db",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Due to the rapid sequence evolution of viral genomes, sequence-based approaches can often fail to detect similarity between distantly evolutionarily related genes. However, protein structure can remain conserved even when sequences diverge dramatically ",
				},
				{
					type: "citation",
					attrs: {
						id: "n9a8owxrumz",
						href: null,
						value: "https://doi.org/10.1111/brv.12905",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ". For this reason, we next turned to structure-based homology searches to explore whether viral capsids may be present in additional species.",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "n8a8m06f97t",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "To search for structural homologs of capsid proteins in venoms, we first used Foldseek, a tool that enables structure-based searches for similar proteins ",
				},
				{
					type: "citation",
					attrs: {
						id: "nbda2j9zxgi",
						href: null,
						value: "https://doi.org/10.1038/s41587-023-01773-0",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ". We performed structure-based searches on three representative capsid structural proteins (not the RdRps) identified above — the ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ixodes ",
				},
				{
					type: "text",
					text: "rhabdovirus nucleocapsid (KAG0427517), the ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ixodes ",
				},
				{
					type: "text",
					text: "sabavirus nucleocapsid (EEC17452), and the ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ixodes",
				},
				{
					type: "text",
					text: " Gag-Pol protein (XP_042148722) — as well as the bracovirus major capsid protein from the parasitoid wasp ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Microplitis demolitor",
				},
				{
					type: "text",
					text: " (vp39, NP_001401748).",
				},
			],
		},
		{
			type: "blockquote",
			attrs: {
				id: "nzvm6ibnm3c",
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "paragraph",
					attrs: {
						id: "ne6ufhxuu5y",
						class: null,
						textAlign: null,
						rtl: null,
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
							],
							text: "Dealing with disorder in structure comparison",
						},
					],
				},
				{
					type: "paragraph",
					attrs: {
						id: "n4gfiojdh06",
						class: null,
						textAlign: null,
						rtl: null,
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "text",
							text: "One thing to note is that these proteins all contain disordered regions, which complicates structural interpretation. We get an idea of the amount of disorder in a protein using the pLDDT (predicted local distance difference test), which is calculated by AlphaFold and gives an estimate of the amount of confidence for the structure on a per-residue basis ",
						},
						{
							type: "citation",
							attrs: {
								id: "nd1u8ea983s",
								href: null,
								value: "https://doi.org/10.1093/bioinformatics/btt473",
								unstructuredValue: "",
								customLabel: "",
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
						},
						{
							type: "text",
							text: ". To get an idea of the total amount of disorder for the protein, we average the per-residue pLDDT scores. Regions of proteins that are well-ordered will have a score between 90–100, regions with a score between 70–90 have some structure but have less confidence, regions between 50–70 are generally low-confidence, and regions with a pLDDT of less than 50 are likely disordered ",
						},
						{
							type: "citation",
							attrs: {
								id: "n0j2a9gj96f",
								href: null,
								value: "https://doi.org/10.1101/2021.09.26.461876",
								unstructuredValue: "",
								customLabel: "",
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
						},
						{
							type: "text",
							text: ". KAG0427517 had an average pLDDT of 62; EEC17452 had an average pLDDT of 77; NP_001401748 had an average pLDDT of 72; and XP_042148722 had an average pLDDT of 78. Despite the disorder throughout these proteins, we did get hits that align well with the structured regions of the proteins.",
						},
					],
				},
			],
		},
		{
			type: "image",
			attrs: {
				id: "n41kwh0e03y",
				url: "https://assets.pubpub.org/2tvwltfs/Fig5_Dist.-structural-homologs-71721330929243.png",
				size: 85,
				align: "center",
				caption:
					'<p id="n9kw5z7t31e" data-reactroot=""><strong>Distribution of structural homologs of endogenized viral proteins from ticks across the AlphaFold database</strong>.</p><p id="nw63ly6vnca" data-reactroot="">We used Foldseek to identify structural homologs of:</p><p id="npidir78l5x" data-reactroot="">(A) the tick sabavirus N protein (EEC17452.1)</p><p id="npjjdnkwd8m" data-reactroot="">(B) the tick rhabdovirus N protein (KAG0427517.1)</p><p id="n1cdldu58kp" data-reactroot="">(C) the tick Gag-Pol protein (XP_042148722.1). In this panel, we’ve collapsed all non-animal clades. </p><p id="nz2ku8o7l0o" data-reactroot="">We show evolutionary relationships between the species with Foldseek hits to each protein using phylogenetic trees. In this case, every species on the tree has at least one hit to the query tick protein. Clades of animals with parasitic or venomous lifestyles are indicated with brown highlights around leaf tips and name, whereas non-venomous and non-parasitic organisms are shown with white highlighting.</p><p id="nni9lfn8bl2" data-reactroot=""><br/>Newick files for trees shown in this figure can be found <a href="https://zenodo.org/records/12775362" target="_blank">here</a>. In panel C, we couldn’t show individual species names due to the size and complexity of the tree. However, information about all hits for each query, including their species of origin, can be found <a href="https://zenodo.org/records/12775362/files/Foldseek_capsid_followup_05092024.xlsx?download=1" target="_blank">here</a>.</p>',
				altText: "",
				hideLabel: false,
				fullResolution: false,
				href: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
		},
		{
			type: "heading",
			attrs: {
				level: 3,
				fixedId: "",
				id: "sabavirus-n-hits-eec174521",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Sabavirus N hits (EEC17452.1)",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nb93w8xhopd",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "The ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "I. scapularis",
				},
				{
					type: "text",
					text: " nucleocapsid protein EEC17452.1 showed homology to six tick proteins and one spider protein (",
				},
				{
					type: "reference",
					attrs: {
						id: "nocpzf6uloh",
						targetId: "n41kwh0e03y",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ", A). We found the original EEC17452.1 protein plus one additional ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "I. scapularis",
				},
				{
					type: "text",
					text: " hit. We also had hits in ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "I. ricinus",
				},
				{
					type: "text",
					text: "; the soft tick, ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ornithodros erraticus",
				},
				{
					type: "text",
					text: "; and the orb-weaver spider, ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Araneus ventricosus.",
				},
			],
		},
		{
			type: "heading",
			attrs: {
				level: 3,
				fixedId: "",
				id: "rhabdovirus-n-hits-kag04275171",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Rhabdovirus N hits (KAG0427517.1)",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "ng6l037q1wb",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "We found structural homologs of the ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ixodes persulcatus ",
				},
				{
					type: "text",
					text: "rhabdovirus nucleocapsid protein KAG0427517 protein in the ticks",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: " Ixodes scapularis",
				},
				{
					type: "text",
					text: " and ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Rhipicephalus microplus",
				},
				{
					type: "text",
					text: ", as well as several other species with intriguing parallels to ticks (",
				},
				{
					type: "reference",
					attrs: {
						id: "nb1p4nuu9rg",
						targetId: "n41kwh0e03y",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ", B). Namely, we found structural homologs in other hematophagous (blood-sucking) species — mosquitos (",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Aedes aegypti",
				},
				{
					type: "text",
					text: " and ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Anopheles darlingi",
				},
				{
					type: "text",
					text: ") and bed bugs (",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Cimex lectularius",
				},
				{
					type: "text",
					text: "). Intriguingly, we also found homologs in two sap-sucking species — the aphids ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Aphis craccivora",
				},
				{
					type: "text",
					text: " and ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Sipha flava",
				},
				{
					type: "text",
					text: ". We also found homologs in the plant parasitic spider mite,",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: " Tetranychus urticae",
				},
				{
					type: "text",
					text: ", and the human parasitic helminth worm, ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Enterobius vermicularis",
				},
				{
					type: "text",
					text: ", also known as pinworm. This pattern is compelling because all of these species, including ticks, have parasitic interactions with their host species, suggesting that the viral capsid may play a role in diverse types of parasitic lifestyles. The only non-parasites that had structural homologs of these proteins were the diamondback moth, ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Plutella xylostella,",
				},
				{
					type: "text",
					text: " and the speckled wood butterfly, ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Pararge aegeria",
				},
				{
					type: "text",
					text: ". Interestingly, ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "P. xylostella",
				},
				{
					type: "text",
					text: " is a natural host of several parasitoid wasps, suggesting that the nucleocapsid protein could be the result of parasite-to-host HGT. The significance of the N protein homolog in ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "P. aegeria ",
				},
				{
					type: "text",
					text: "is unknown.",
				},
			],
		},
		{
			type: "heading",
			attrs: {
				level: 3,
				fixedId: "",
				id: "gag-pol-hits-xp0421487221",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Gag-Pol hits (XP_042148722.1)",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nlihn7sj1v3",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "The tick retrotransposon Gag-Pol protein XP_042148722.1 returned 924 hits in our Foldseek search. These hits are widely distributed across eukaryotes, including many parasitic blood and sap-feeding species (",
				},
				{
					type: "reference",
					attrs: {
						id: "np6inc2orli",
						targetId: "n41kwh0e03y",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ", C). This Foldseek search also identified Arc1 (3.69 × 10",
				},
				{
					type: "text",
					marks: [
						{
							type: "sup",
						},
					],
					text: "−8",
				},
				{
					type: "text",
					text: ") and Arc2 (2.00 × 10",
				},
				{
					type: "text",
					marks: [
						{
							type: "sup",
						},
					],
					text: "−5",
				},
				{
					type: "text",
					text: ") from ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Drosophila melanogaster",
				},
				{
					type: "text",
					text: " as structural homologs of the tick gal-pol protein. Drosophila Arc proteins are domesticated viral capsids that participate in RNA delivery between cells ",
				},
				{
					type: "citation",
					attrs: {
						id: "nf1f1mqws1r",
						href: null,
						value: "https://doi.org/10.1016/j.cell.2017.12.024",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ".",
				},
			],
		},
		{
			type: "heading",
			attrs: {
				level: 3,
				fixedId: "",
				id: "bracovirus-vp39-hits-np001401748",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Bracovirus vp39 hits (NP_001401748)",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nlko0k1q33z",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "The only structural homologs we found of the bracovirus vp39 protein NP_001401748 are from other parasitoid wasps, ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Cotesia chilonis",
				},
				{
					type: "text",
					text: " and ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Chelonus inanitus",
				},
				{
					type: "text",
					text: ".",
				},
			],
		},
		{
			type: "heading",
			attrs: {
				level: 2,
				fixedId: "",
				id: "co-clustering-venom-proteins-with-known-viral-capsids-reveals-a-putative-uncharacterized-eukaryotic-viral-capsid",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Co-clustering venom proteins with known viral capsids reveals a putative, uncharacterized eukaryotic viral capsid",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "naekhpd2fh3",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "We next wanted to see if we could extend structural homology-based approaches to whole-venom transcriptomes. Because all of our prior results highlighted the presence of potential capsids in tick species, we chose to focus on proteins from tick salivary transcriptomes for this analysis. We analyzed structures from ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Ornithodoros turicata",
				},
				{
					type: "text",
					text: ", a species of soft-bodied tick in the family ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "Argasidae.",
				},
				{
					type: "text",
					text: " We aggregated folded protein structures from ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "O. turicata",
				},
				{
					type: "text",
					text: " and folded protein structures from known capsid proteins from the VOG database. We then performed clustering to see if any capsids would cluster with proteins from tick saliva. We identified 47 structural clusters (Figure 6, A). From this starting point, we could explore the candidate list of clusters for possible evidence of viral proteins clustering with venom proteins. We see the majority of clusters are quite specific either to viruses or to ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "O. turicata",
				},
				{
					type: "text",
					text: ", but there were six candidate clusters containing both a mix of proteins from the two datasets and a high degree of structural homology (TM > 0.5).",
				},
			],
		},
		{
			type: "image",
			attrs: {
				id: "nult3fgq8zx",
				url: "https://assets.pubpub.org/l9sytd93/Fig6_PC-tick-proteins-51721331036541.png",
				size: 100,
				align: "center",
				caption:
					'<p id="ncdnnipu0mu" data-reactroot=""><strong>ProteinCartography analysis of viral capsid proteins with </strong><strong><em>Ornithodoros turicata </em></strong><strong>proteins reveals two tick proteins that cluster with capsid proteins</strong>.</p><p id="r39331335" data-reactroot="">We clustered all <em>O. turicata </em>structures available on UniProt with ESMFold predictions of the capsid proteins in the VOG database using ProteinCartography.</p><p id="r3421465495" data-reactroot="">(A) A UMAP of co-clustered tick and viral proteins. Here, each protein is a single point on the UMAP and is colored by cluster membership.</p><p id="r7148496963" data-reactroot="">(B) The same UMAP as in (A), but here tick proteins are green, and viral proteins are grey. Generally, the capsid proteins and tick proteins are clustered separately). We identified cluster 22 (boxed) as a cluster that contained two tick proteins alongside proteins from four different viral capsid VOGs.</p><p id="r3770214916" data-reactroot="">(C) Zoom-in on cluster 22. Tick proteins A0A2R5L5R2 and A0A2R5L7H1 cluster with VOG proteins, some of which are annotated as mimivirus capsids.</p>',
				altText: "",
				hideLabel: false,
				fullResolution: true,
				href: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
		},
		{
			type: "paragraph",
			attrs: {
				id: "nkpq5eobm90",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "We followed up on cluster 22, which contained tick proteins A0A2R5L5R2 and A0A2R5L7H1, along with proteins annotated as “major capsid protein” from mimivirus (",
				},
				{
					type: "reference",
					attrs: {
						id: "nzdv88r9f80",
						targetId: "nult3fgq8zx",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ", B). Mimiviruses are giant viruses that typically infect species of amoeba ",
				},
				{
					type: "citation",
					attrs: {
						id: "ngbl4tmfhsq",
						href: null,
						value: "https://doi.org/10.1126/science.1081867",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ". However, the mimivirus capsid protein is in the same protein family as the capsid from African swine fever virus, which ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "O. turicata",
				},
				{
					type: "text",
					text: " ticks transmit. By sequence, the ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "O. turicata",
				},
				{
					type: "text",
					text: " proteins found in this cluster are highly divergent from capsids from both the mimivirus and African swine fever virus. Unfortunately, there are no ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "O. turicata",
				},
				{
					type: "text",
					text: " genomes available, so it’s not currently possible to see if this protein comes from an unknown contaminating virus or is endogenous to the tick genome.",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nenv3q0wzfq",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "After our initial test with ",
				},
				{
					type: "text",
					marks: [
						{
							type: "em",
						},
					],
					text: "O. turicata",
				},
				{
					type: "text",
					text: ", we didn't pursue this clustering approach further. This method is computationally challenging to scale outside of a few salivary transcriptomes, and our ability to interpret the hits that come out of this approach is limited.",
				},
			],
		},
		{
			type: "heading",
			attrs: {
				level: 1,
				fixedId: "",
				id: "key-takeaways",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Key takeaways",
				},
			],
		},
		{
			type: "bullet_list",
			attrs: {
				id: "nmd75x9j2xs",
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "list_item",
					attrs: {
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "paragraph",
							attrs: {
								id: "ni4utg7mbrf",
								class: null,
								textAlign: null,
								rtl: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "text",
									text: "We detected putative endogenized viral capsids encoded by ticks and several other parasitic species, and the tick-encoded viral capsids are expressed in tick saliva. ",
								},
							],
						},
					],
				},
				{
					type: "list_item",
					attrs: {
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "paragraph",
							attrs: {
								id: "nm6lvf7yhp7",
								class: null,
								textAlign: null,
								rtl: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "text",
									text: "Our highest-confidence tick-encoded capsid types are RNA nucleocapsids and retrotransposon Gag-Pol proteins. We predict that both interact with and help package RNA molecules.",
								},
							],
						},
					],
				},
				{
					type: "list_item",
					attrs: {
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "paragraph",
							attrs: {
								id: "nh13etjnplo",
								class: null,
								textAlign: null,
								rtl: null,
								suggestionId: null,
								suggestionTimestamp: null,
								suggestionUserId: null,
								suggestionDiscussionId: null,
								suggestionKind: null,
								suggestionOriginalAttrs: null,
							},
							content: [
								{
									type: "text",
									text: "These results suggest the possibility that some parasites use packaged nucleic acids to interact with their hosts.",
								},
							],
						},
					],
				},
			],
		},
		{
			type: "heading",
			attrs: {
				level: 1,
				fixedId: "",
				id: "discussion",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Discussion",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "n4lx8zz6nob",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Venomous species have evolved an enormous biological arsenal of host-manipulating factors. This particular study was inspired by the ancient use of viral capsids for host gene delivery by parasitoid wasps. We took a multi-pronged search across venomous species, looking for examples where other venomous species may have evolved the use of viral capsids. The two major categories of viral capsids we identified originate from RNA viruses [rhabdovirus and sabavirus (a type of bunyavirus)] and the retrotransposon Gag-Pol protein from LTR retrotransposons.",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nlfu4y6pc8y",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "We were excited to detect these capsid-like proteins outside of parasitoid wasps (",
				},
				{
					type: "reference",
					attrs: {
						id: "n05at2c4ryx",
						targetId: "njp74m066hd",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: "). Taxonomic analysis of these species suggests that some endogenized capsids are enriched in organisms that have parasitic interactions with their hosts. In particular, several tick species appear to have homologs of multiple types of viral capsids. These appear to have been endogenized in the ticks’ genomes and expressed in their saliva, the tick equivalent of venom. Integration of viral sequences into tick genomes has been observed before ",
				},
				{
					type: "citation",
					attrs: {
						id: "ne1hy6u8qbd",
						href: null,
						value: "https://doi.org/10.3389/fcimb.2013.00025",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "citation",
					attrs: {
						id: "nk04jpan6h6",
						href: null,
						value: "https://doi.org/10.1093/ve/vez010",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "citation",
					attrs: {
						id: "nhplnsy2ytb",
						href: null,
						value: "https://doi.org/10.1101/2022.08.17.504163",
						unstructuredValue: "",
						customLabel: "",
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
				},
				{
					type: "text",
					text: ". Hard ticks feed on hosts for days to weeks, unlike the rapid predator-prey interactions found in many venomous species, such as snakes. This extended host-parasite interaction could be analogous to the extended interaction of parasitoid wasps and lepidopteran hosts. This sustained interaction likely requires modes of host manipulation that act over longer timescales — gene delivery may be one of these.",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nn0ufcwtevs",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Our findings suggest that the endogenization of viral capsids by venomous and/or parasitic species may be more widespread than previously understood, and these endogenized capsids are common in parasitic species that have extended interactions with their hosts. Right now, we don’t know the physiological role, if any, of these endogenized capsids in the parasitic lifestyle. However, we believe our findings are an intriguing starting point for further study into the specific roles of these capsids in host–parasite biology and may suggest novel approaches to molecular delivery in mammals.",
				},
			],
		},
		{
			type: "heading",
			attrs: {
				level: 1,
				fixedId: "",
				id: "next-steps",
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Next steps",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "n42nnaxb0xj",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "text",
					text: "Our initial work has established that capsid-like proteins are found in a variety of parasitic species, including some that bite humans. The biological significance of these endogenized capsids is still unknown. For us, the most translationally exciting outcome would be that capsids from human parasites are participating in gene delivery to human cells. Future work should focus on understanding whether any of these organisms actually use these capsids for nucleic acid delivery and, if so, what specific cargo they deliver.",
				},
			],
		},
		{
			type: "paragraph",
			attrs: {
				id: "nihj4yai9qz",
				class: null,
				textAlign: null,
				rtl: null,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
		},
		{
			type: "horizontal_rule",
			attrs: {
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
		},
		{
			type: "iframe",
			attrs: {
				id: "nkbbuug3q2z",
				url: "https://publishing-tools.arcadiascience.com/typeform/AacF8YKM?pub_name=rec581s2eybk6I5Cw",
				size: 100,
				height: 425,
				align: "center",
				caption: "",
				hideLabel: false,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
		},
		{
			type: "blockquote",
			attrs: {
				id: "n4mp6p82h92",
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
			content: [
				{
					type: "paragraph",
					attrs: {
						id: "nq5xgpyt2xh",
						class: null,
						textAlign: null,
						rtl: null,
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "text",
							text: "Share your thoughts!",
						},
					],
				},
				{
					type: "paragraph",
					attrs: {
						id: "nz00o4ipzt9",
						class: null,
						textAlign: null,
						rtl: null,
						suggestionId: null,
						suggestionTimestamp: null,
						suggestionUserId: null,
						suggestionDiscussionId: null,
						suggestionKind: null,
						suggestionOriginalAttrs: null,
					},
					content: [
						{
							type: "text",
							marks: [
								{
									type: "strong",
								},
								{
									type: "link",
									attrs: {
										href: "https://bit.ly/3xo7dkm",
										title: null,
										target: "_blank",
										pubEdgeId: null,
									},
								},
							],
							text: "Watch a video tutorial",
						},
						{
							type: "text",
							text: " on making a PubPub account and commenting. Feel free to add line-by-line comments anywhere within this text, provide overall feedback by commenting in the box at the bottom of the page, or post about this work on social media. Please make all feedback public so other readers can benefit from the discussion.",
						},
					],
				},
			],
		},
		{
			type: "iframe",
			attrs: {
				id: "nk936e7tnye",
				url: "https://publishing-tools.arcadiascience.com/twitter/398034609045805608",
				size: 100,
				height: 500,
				align: "center",
				caption: "",
				hideLabel: false,
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
		},
		{
			type: "horizontal_rule",
			attrs: {
				suggestionId: null,
				suggestionTimestamp: null,
				suggestionUserId: null,
				suggestionDiscussionId: null,
				suggestionKind: null,
				suggestionOriginalAttrs: null,
			},
		},
	],
} as Record<string, any>;

export const pubBodyHTML = `<div class="editor read-only ProseMirror" contenteditable="false" translate="no"><h1 id="purpose">Purpose</h1><p id="n33ucq2qaha">The development of AAV capsids for therapeutic gene delivery has exploded in popularity over the past few years. However, humans aren’t the first or only species using viral capsids for gene delivery — wasps evolved this tactic over 100 million years ago. Parasitoid wasps that lay eggs inside arthropod hosts have co-opted ancient viruses for gene delivery to manipulate multiple aspects of the host’s biology, thereby increasing the probability of survival of the wasp larvae <span id="n67l65xpyip" data-node-type="citation" data-value="https://doi.org/10.1016/j.virusres.2006.01.001" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="n67l65xpyip-note-popover" contenteditable="false">[1]</span><span id="n2piklt9xg9" data-node-type="citation" data-value="https://doi.org/10.1016/j.tim.2004.10.004" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="n2piklt9xg9-note-popover" contenteditable="false">[2]</span>.&nbsp;</p><p id="nucymbbl1uc">We wondered if venomous species that bite humans have also evolved to use viral capsids to deliver molecules that manipulate their hosts. We used a multi-pronged sequence- and structure-based search for viral capsids across venomous species and found evidence for endogenized viral capsids, most notably in ticks. Though we cast a broad net in this effort, we were most interested in finding novel nucleic acid delivery systems in species that deliver cargo to humans, as these would be the most useful therapeutic modalities. Here, we focused on finding endogenized capsid proteins, and our follow-up work has focused on developing methods to specifically identify potential nucleic acid cargos in parasites. More to come on this in a follow-up pub!</p><p id="nepq27n6yik">These findings may be useful to other scientists interested in the domestication of viral genes, especially in the context of parasitism.</p><ul id="nxagub6a2yz"><li><p id="njuywwe9ie7"><strong>Data </strong>from our capsid HMM search and our preHGT run to find putatively endogenized viral capsids are available on <a href="https://zenodo.org/records/12775362" target="_blank">Zenodo</a>, as is our follow-up work looking into individual hits with BLAST searches of tick salivary transcriptome, genomic neighborhood analysis, and Foldseek analysis.</p></li><li><p id="n3ed7v01mlb"><strong>Data </strong>from our ProteinCartography runs, including structures of all tick and viral capsid proteins, the configuration file, and all outputs, are available on <a href="https://zenodo.org/records/12796464" target="_blank">Zenodo</a>. We performed ProteinCartography analysis using version 0.4.2 of the pipeline, found in <a href="https://github.com/Arcadia-Science/ProteinCartography/tree/v0.4.2" target="_blank">this GitHub repository</a>.</p></li><li><p id="n4teo6c9gk4">All associated <strong>code</strong> is available in this <a href="https://github.com/Arcadia-Science/capsids/tree/pubv1.0" target="_blank">GitHub repository</a>.</p></li></ul><blockquote id="nijip1ieibq"><p id="nkjq02met0t">Share your thoughts!</p><p id="n7u3k403rps"><strong><a href="https://bit.ly/3xo7dkm" target="_blank">Watch a video tutorial</a></strong> on making a PubPub account and commenting. Feel free to add line-by-line comments anywhere within this text, provide overall feedback by commenting in the box at the bottom of the page, or post about this work on social media. Please make all feedback public so other readers can benefit from the discussion.</p></blockquote><h1 id="weve-put-this-effort-on-ice">We’ve put this effort on ice! 🧊</h1><blockquote id="npj96iygw1a"><p id="n9q55683k02"><strong>#DeadEnd</strong></p><p id="ndkwbii73ia">We found intriguing capsid-like proteins across several parasitic species. To advance, we'd need to figure out what cargo the putative capsids deliver, if any. In a follow-up study, we tried to computationally identify packaged nucleic acids in parasites, as we’re most interested in therapeutic nucleic acid delivery, but our results weren’t sufficiently compelling to warrant further investment.</p><p id="n4qmfzify47"><a href="https://research.arcadiascience.com/icebox" target="_blank"><sup>Learn more</sup></a><sup> about the Icebox and the different reasons we ice projects.</sup></p></blockquote><h1 id="background-and-goals">Background and goals</h1><p id="nobsrjekjzw">Capsid-based delivery of therapeutics represents a growing modality of interest across the biotech/pharma industry. The focus of most recent efforts has been on a specific class of capsids from adenovirus-associated viruses (AAVs). AAV-based therapies have been approved for use in humans, and researchers are engineering and applying these vectors for a variety of indications <span id="n05riw93ovg" data-node-type="citation" data-value="https://doi.org/10.1038/s41573-019-0012-9" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="n05riw93ovg-note-popover" contenteditable="false">[3]</span>. However, co-opting viral capsids for gene delivery is an ancient innovation, dating back over 100 million years.</p><figure id="njp74m066hd" data-node-type="image" data-size="85" data-align="center" data-url="https://assets.pubpub.org/39g5lkfx/Fig1_Parasitoid-wasp-GA-31721329603673.png" data-caption="<p id=&quot;n9eqg1k4tzp&quot; data-reactroot=&quot;&quot;><strong>Braconid parasitoid wasps use domesticated viruses to protect their eggs from the host immune system</strong>.</p><p id=&quot;r6807458188&quot; data-reactroot=&quot;&quot;>Braconid parasitoid wasps like <em>Microplitis demolitor</em> lay their eggs in living lepidopteran hosts, where the larvae consume the host as a food source throughout development. To protect their eggs from the caterpillar immune response, the wasps evolved a fascinating strategy of deploying an ancient endogenized betanudivirus (called a bracovirus) as a gene delivery agent. The wasps inject viral particles containing wasp DNA alongside their eggs, and prolonged expression of these genes in the caterpillar alters the host immune system to protect the developing larvae.</p><p id=&quot;r6313380962&quot; data-reactroot=&quot;&quot;>mya: Million years ago</p>" data-alt-text="" contenteditable="false"><img srcset="" src="https://assets.pubpub.org/39g5lkfx/Fig1_Parasitoid-wasp-GA-31721329603673.png" alt="" aria-labelledby="njp74m066hd-figure-caption"><figcaption id="njp74m066hd-figure-caption"><div><strong spellcheck="false">Figure 1</strong><div><p id="n9eqg1k4tzp" data-reactroot=""><strong>Braconid parasitoid wasps use domesticated viruses to protect their eggs from the host immune system</strong>.</p><p id="r6807458188" data-reactroot="">Braconid parasitoid wasps like <em>Microplitis demolitor</em> lay their eggs in living lepidopteran hosts, where the larvae consume the host as a food source throughout development. To protect their eggs from the caterpillar immune response, the wasps evolved a fascinating strategy of deploying an ancient endogenized betanudivirus (called a bracovirus) as a gene delivery agent. The wasps inject viral particles containing wasp DNA alongside their eggs, and prolonged expression of these genes in the caterpillar alters the host immune system to protect the developing larvae.</p><p id="r6313380962" data-reactroot="">mya: Million years ago</p></div></div></figcaption></figure><p id="njbuy8jyy9p">Scientists uncovered this innovation by studying parasitoid wasps, which use viral capsids derived from ancient polydnaviruses to deliver genes to their caterpillar hosts (<a class="reference" data-node-type="reference" data-target-id="njp74m066hd" href="#njp74m066hd" contenteditable="false">Figure 1</a>) <span id="nq3mf22frzr" data-node-type="citation" data-value="https://doi.org/10.1038/s42003-020-01623-8" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="nq3mf22frzr-note-popover" contenteditable="false">[4]</span><span id="n271lwt830j" data-node-type="citation" data-value="https://doi.org/10.1016/j.coviro.2017.07.002" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="n271lwt830j-note-popover" contenteditable="false">[5]</span>. While there is an enormous diversity of ecologies and lifestyles across parasitoid wasps, many parasitize other arthropods, especially lepidopterans. These wasps inject their eggs into caterpillars, and the developing larvae use the host as a source of food. To help protect the eggs, the parasitoid wasps co-inject viral-like particles that manipulate the caterpillar’s biology, including its immune system <span id="n4h9efgzc3i" data-node-type="citation" data-value="https://doi.org/10.3389/fimmu.2022.877027" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="n4h9efgzc3i-note-popover" contenteditable="false">[6]</span>. The injected packages are made up of viral capsids filled with wasp DNA that helps to protect the wasps’s larvae. Once injected into the host, wasp DNA is integrated into the host genome to drive expression of host-manipulating factors, such as immune-dampening proteins.</p><p id="n0n6b4uf95d">Wasps have co-opted distinct types of polydnaviruses multiple times <span id="n1pek8zaqhp" data-node-type="citation" data-value="https://doi.org/10.1098/rstb.2013.0051" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="n1pek8zaqhp-note-popover" contenteditable="false">[7]</span>, with different wasp lineages harboring distinct endogenized viruses that they deploy for host manipulation. Of these, some of the best-studied are the bracoviruses found in the genomes of braconid wasps like <em>Microplitis</em> and <em>Cotesia. </em>Although the bracovirus capsid proteins play a key role in delivery of nucleic acid cargo to host species, studies on bracovirus biology show key roles for other virus-derived machinery in the process — especially packaging machinery and envelope proteins.</p><p id="nzutgw5cgsj">Our goal in this pilot was to explore whether other venomous species have similarly evolved the ability to use viral capsids for molecular delivery. Though we searched for endogenized capsids of any sort, we’d hoped to identify capsids that deliver genes to a human host. If we can find such capsids, they might serve as new starting points for developing gene delivery systems.</p><p id="nkwak1lq67y">We used a combination of computational approaches to scan for the presence of capsid-like proteins across venoms, including sequence-, HGT-, and structure-based searches. We ultimately found capsid-like genes in the venom-like saliva of ticks as well as in the genomes of other parasitic species, including parasitic nematodes (helminths), mosquitoes, and aphids. Jump straight to <a href="#the-results">these results</a> or read on to learn the details of our approach.</p><h1 id="the-approach">The approach</h1><p id="nxan7acu8f8">To identify capsid-like proteins in venomous species, we applied two strategies: a <strong>sequence homology-based search</strong> for capsids in venom gland transcriptomes (HMM-based detection) and our <strong>horizontal gene transfer (HGT) detection pipeline</strong> to detect genes of putative viral origin in genomes of venomous species (preHGT). We then followed up on the most promising hits by searching for structural homologs across the AlphaFold database using Foldseek. Last, we experimented with a structural clustering-based approach to identify capsid-like structures in salivary transcriptomes but didn’t pursue this approach extensively as it wasn’t scalable and results were hard to interpret.</p><h2 id="an-hmm-based-search-of-venom-transcriptomes-for-viral-capsids">An HMM-based search of venom transcriptomes for viral capsids</h2><p id="n9ywpb2u23u">We first performed a sequence-based search for capsid-like proteins in venom transcriptomes using hidden Markov models (HMMs) of viral capsids. A curated database of viral protein families and associated HMM profiles for each family is available for download from the <a href="https://vogdb.org/" target="_blank">viral orthologous groups (VOGs) database</a> (version vog217). In addition, because bracovirus capsids aren't included in the VOG database, we also generated a custom HMM profile to include in searches based on a PSI-BLAST search results using the <em>Microplitis demolitor</em> major bracovirus capsid protein vp39 as a seed sequence.</p><p id="nbfinv30fl8">We then used these HMMs to search against 156 transcriptomes from 135 species using HMMER3 <span id="ncs4jrwy8dn" data-node-type="citation" data-value="https://doi.org/10.1093/nar/gkr367" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="ncs4jrwy8dn-note-popover" contenteditable="false">[8]</span>. To maximize our ability to find distant homologs, we initially returned all hits from the HMMER search with no E-value cutoff and followed up with manual filtering steps.</p><blockquote id="njkopr46hh7"><p id="nir143deub7">You can find the <strong>list of species</strong> included in this database, along with all <strong>scripts, metadata, and results</strong>, in <a href="https://github.com/Arcadia-Science/capsids/tree/pubv1.0" target="_blank">this GitHub repository</a>.</p></blockquote><h3 id="filtering-results-from-the-hmm-based-search">Filtering results from the HMM-based search&nbsp;</h3><p id="nd4c0abur24">We first filtered results to include only hits from a subset of VOGs containing the word “capsid,” but not “capsid maturation protease” or “capsid assembly protease,” in their annotation, as we found these VOGs don’t encode actual capsid proteins themselves, and were driving a lot of spurious hits to animal proteases. Next, we manually curated the output file in an attempt to further filter out false positives. This analysis suffered from three major sources of false positives: hits from non-endogenized viruses that are part of the venom virome, hits to venom proteins that don't appear to be endogenized capsids but share a common domain with the capsid HMM (for instance, a protease domain or an immunoglobulin domain), and spurious hits to the animal proteome due the fact we didn't use any E-value cutoff for our initial HMMER search.&nbsp;</p><p id="nhecxvatzqb">To identify false positives, we performed BLASTp on the top 210 hits from the HMMER3 search (E-value 1.20 × 10<sup>−81</sup>–0.0026). We performed these searches using the NCBI web server in March of 2023. If the protein had close hits to viruses but no eukaryote genomes, we decided that it could be a contaminant virus and not endogenized. If the protein had hits to proteins from other eukaryotic genomes but was consistently annotated to encode a non-capsid protein, we considered it to be a false positive. Some predicted proteins repeatedly came up as false-positive hits, which likely reflects a shared domain with our viral HMMs. Common hits that fell into this category were Clp protease, ovarian tumor domain (OTU), LRR domain, IG domain, hemicentin, obscurin, peroxidasin, titin, twitchin, fasciclin, neural cell adhesion molecule, roundabout, AAA ATPase, and 26S proteasome subunit. After manual curation, we were left with 14 putative endogenized capsids (not counting our expected, positive-control hit — the vp39 HMM to the <em>Microplitis demolitor</em> major bracovirus capsid protein vp39). The highest-confidence endogenized capsids we recovered were from four parasitoid wasp species and one spider.<br><br class="ProseMirror-trailingBreak"></p><blockquote id="n382bsbwgaq"><p id="nikssfcf7xz">Download our <strong>manually annotated results</strong> from searching VOG HMMs against venom transcriptomes on <a href="https://zenodo.org/records/12775362/files/Annotated_capsid_no_cutoffs_summary_with_seqs_05092024.xlsx?download=1" target="_blank">Zenodo</a>.</p></blockquote><h2 id="horizontal-gene-transfer-hgt-analysis-of-venomous-animal-genomes">Horizontal gene transfer (HGT) analysis of venomous animal genomes&nbsp;</h2><p id="nkrxfjdztwe">Because the evolutionary acquisition of viral genes is a key example of inter-kingdom horizontal gene transfer (HGT), we decided to also deploy an approach that was specifically tailored to detect inter-kingdom HGT events. We used our preHGT <span id="n9ccze633a9" data-node-type="citation" data-value="https://doi.org/10.57844/ARCADIA-JFBP-7P11" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="n9ccze633a9-note-popover" contenteditable="false">[9]</span> pipeline to scan genus-level pangenomes for recent viral HGT events using a BLAST-based taxonomic approach. <span class="discussion-range d-77485d33-f719-46aa-bbc1-86a8b5a1cf47">We used gene models for nine venomous species: </span><em><span class="discussion-range d-77485d33-f719-46aa-bbc1-86a8b5a1cf47">Ixodes scapularis, Ixodes persulcatus, Microplitis demolitor, Sepia pharaonis, Trichomalopsis sarcophagae, Ampulex compressa, Ophiophagus hannah, Bothrops jararaca, </span></em><span class="discussion-range d-77485d33-f719-46aa-bbc1-86a8b5a1cf47">and </span><em><span class="discussion-range d-77485d33-f719-46aa-bbc1-86a8b5a1cf47">Naja naja</span></em><span class="discussion-range d-77485d33-f719-46aa-bbc1-86a8b5a1cf47">.</span><span class="discussion-mount dm-77485d33-f719-46aa-bbc1-86a8b5a1cf47 ProseMirror-widget" contenteditable="false" style="display: inline;"><span class="discussion-group-component" tabindex="-1" style="left: 830.172px; width: 292.336px;"><span class="discussion-nav-component" style="color: rgb(72, 75, 80); border-bottom: 1px solid transparent;"><style>.d-undefined, .lh-undefined { background-color: rgba(0, 0, 0, 0.2) !important; }</style><span class="discussion-bubble-component" role="button" tabindex="0"><span class="bubble" style="border: 1px solid rgb(72, 75, 80);">2</span><span class="label">Cyril Eleftheriou, Adair L. Borges</span></span></span></span></span><img class="ProseMirror-separator" alt=""><br class="ProseMirror-trailingBreak"></p><p id="n9kgncywlc4">We created nucleotide pangenomes at the genus level by clustering genes at 90% length and sequence identity to determine the unique set of genes for the genus. We then chose a representative sequence for each cluster by selecting the sequence with the most alignments. We used BLASTp to compare representative proteins to a clustered non-redundant (nr) database, inspired by NCBI’s <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2022/05/02/clusterednr_1/" target="_blank">ClusteredNR</a>. The clustered database <span id="n63jrfhg65u" data-node-type="citation" data-value="https://doi.org/10.57844/ARCADIA-W8XT-PC81" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="n63jrfhg65u-note-popover" contenteditable="false">[10]</span> allowed us to capture more taxonomically diverse hits.</p><p id="n1gy0xre13z">We then used the “alien index” to predict whether an HGT event occurred <span id="n42539lubtz" data-node-type="citation" data-value="https://doi.org/10.1126/science.1156407" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="n42539lubtz-note-popover" contenteditable="false">[11]</span>. The alien index is a metric that tells us how similar a given gene is to genes in related, non-self “acceptor” species, compared to unrelated potential “donor” species. In the case of horizontal gene transfer, we should see genes that have high similarity to genes in the donor species and low similarity to genes in species closely related to the acceptor species. We calculate the alien index by subtracting the E-value of the best donor (viral) hit from the E-value of the best non-self acceptor hit. If the best viral hit E-value is closer to zero than the best venomous species hit, the alien index will be positive. We used the following previously published thresholds <span id="nhkughsptrg" data-node-type="citation" data-value="https://doi.org/10.3390/genes8100248" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="nhkughsptrg-note-popover" contenteditable="false">[12]</span> for determining HGT events: an alien index &gt; 0 is possible HGT, &gt;&nbsp;15 is likely HGT, and &gt;&nbsp;45 is highly likely. We removed potential contaminants from this candidate list that we only saw in one genome within an investigated genus and that either had very high (&gt; 90%) identity to a viral sequence or that were very short with high identity (&lt;&nbsp;20,000 bp, &gt;&nbsp;70% identity). We performed ortholog annotation (KEGG, PFAM, and viral and biosynthetic gene clusters) for proteins that may have been horizontally transferred.</p><p id="n1cpuamd4ab">The pipeline produced a table of possible HGT events, including the predicted donor and acceptor taxa, alien index and BLASTp values, ortholog annotations, and genomic location information for this protein in the “acceptor” genome. Each hit is scored based on its likelihood to be either a real HGT event vs. contamination. We filtered out all events that were scored as “likely contamination.” After filtering, we detected 66 putative HGT events, 21 of which were inter-kingdom events predicted to be from a viral donor.&nbsp;</p><p id="n3y2gaamnry">For the six putative viral HGT-derived genes in ticks (<em>Ixodes persulcatus</em> and <em>Ixodes scapularis</em>), we used tBLASTn to see if similar genes were also integrated into the <em>Amblyomma americanum</em> genome, which we hadn’t included in this HGT analysis since we didn't have gene predictions for this species at the time of analysis. We observed multiple hits for KAG0427517.1 and KAG0420414.1 in the <em>A. americanum </em>genome, suggesting that such viral HGT events may be common in ticks.<br><br class="ProseMirror-trailingBreak"></p><blockquote id="nmfl5xgu0w7"><p id="nqyz3q5rgi2">You can find our <strong><a href="https://zenodo.org/records/12775362/files/Annotated_hgt_results_venom_050924.xlsx?download=1" target="_blank">manually annotated results</a></strong> from running preHGT on the genomes of venomous species here and the <strong><a href="https://zenodo.org/records/12775362/files/preHGT_input_genomes.csv?download=1" target="_blank">genomes we input into preHGT</a></strong>, both on <a href="https://zenodo.org/records/12775362" target="_blank">Zenodo</a>.</p></blockquote><blockquote id="n1x4gcoog4v"><p id="nq4xzktvolg">You can find the <strong>preHGT GitHub repo</strong> <a href="https://github.com/Arcadia-Science/prehgt" target="_blank">here</a>.</p></blockquote><h2 id="genomic-context-analysis-of-putative-endogenized-viral-genes">Genomic context analysis of putative endogenized viral genes&nbsp;</h2><p id="nqs9kz6jbe3">To evaluate if viral genes are integrated into the eukaryotic genome (rather than a viral contaminant), we pulled the contigs for each hit from the DBSOURCE field on the protein’s GenBank page. We displayed these contigs using Geneious Prime (version 2022.2.1) and manually analyzed the contigs to assess if they're part of the tick genome by looking for introns, a eukaryote-specific feature, in the capsid gene and predicting origin and function of neighboring genes using the BLAST, HHpred, and Foldseek web servers.</p><blockquote id="nmslu3ry40d"><p id="nvqhpdtlrpi">You can find <strong>GenBank files containing the genomic neighborhood </strong>of putatively horizontally transferred viral genes in the tick genome <a href="https://zenodo.org/records/12775362" target="_blank">here</a>. We’ve shared seven GenBank files on Zenodo —&nbsp;one for each contig, named with the putative viral protein identifier.</p></blockquote><h2 id="salivary-expression-analysis-of-putative-endogenized-viral-genes">Salivary expression analysis of putative endogenized viral genes</h2><p id="nbfrodnx3je">To evaluate whether our viral proteins of interest are expressed in tick saliva, we BLASTed our top seven candidate proteins against <em>Ixodes scapularis and Ixodes ricinus </em>proteins<em> </em>from the NCBI TSA (accession numbers: GADI01P.1, GEGO01P.1, GHXN01.1, GIFC01P.1, and GKHW01.1) using Geneious Prime (version 2022.2.1).</p><blockquote id="nl69qohazd9"><p id="nbvi9vbqzat">Download our <strong>BLAST results</strong> from searching putatively endogenized viral proteins against tick salivary transcriptomes on <a href="https://zenodo.org/records/12775362/files/TSA_blast_results.csv?download=1" target="_blank">Zenodo</a>.</p></blockquote><h2 id="foldseek-analysis-of-putative-endogenized-viral-genes">Foldseek analysis of putative endogenized viral genes</h2><p id="nkd759oktas">To search for structural homologs of our capsid proteins of interest in other species, we used Foldseek, a tool that enables structure-based searches for similar proteins <span id="necgrihm8fa" data-node-type="citation" data-value="https://doi.org/10.1038/s41587-023-01773-0" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="necgrihm8fa-note-popover" contenteditable="false">[13]</span>. We prioritized three of the hits from the HMM- and HGT-based search (<em>Ixodes</em> rhabdovirus nucleocapsid KAG0427517, <em>Ixodes</em> sabavirus nucleocapsid EEC17452, and <em>Ixodes</em> Gag-Pol XP_042148722). We also queried with the wasp bracovirus capsid NP_001401748 that originally inspired this project. There was a structure available in the AlphaFold database for the <em>Ixodes</em> sabavirus nucleocapsid (AF-B7QF30), but not for the other three proteins. We folded the other three using ColabFold (version 1.5.2) <span id="nqslnlzl7sk" data-node-type="citation" data-value="https://doi.org/10.1038/s41592-022-01488-1" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="nqslnlzl7sk-note-popover" contenteditable="false">[14]</span> and used these to search for structures with the Foldseek web tool (version Foldseek 5-53465f0). </p><p id="nbjwocstcf8">We searched all three AlphaFold databases available on Foldseek (i.e., AlphaFold/UniProt50 version 4, AlphaFold/Swiss-Prot version 4, and AlphaFold/Proteome version 4) <span id="npygyt771iu" data-node-type="citation" data-value="https://doi.org/10.1038/s41592-022-01488-1" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="npygyt771iu-note-popover" contenteditable="false">[14]</span><span id="nq3kj45v991" data-node-type="citation" data-value="https://doi.org/10.1093/nar/gkab1061" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="nq3kj45v991-note-popover" contenteditable="false">[15]</span>. One important note is that the AlphaFold database specifically excluded any viral proteins, so any protein hits are likely not coming from the viruses themselves. We then filtered these hits using Foldseek E-values (similar to BLAST E-values), to determine the confidence of their hits. The developers of Foldseek consider E-values less than 0.01 to represent homologous pairs <span id="nmb9dnswga9" data-node-type="citation" data-value="https://doi.org/10.1038/s41586-023-06510-w" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="nmb9dnswga9-note-popover" contenteditable="false">[16]</span>, so we applied that cutoff here as well for the bracovirus capsid, the <em>Ixodes</em> sabavirus nucleocapsid, and the <em>Ixodes</em> rhabdovirus nucleocapsid, which each returned a low number of hits (&lt; 20). However, the <em>Ixodes</em> Gag-Pol protein had a very wide distribution, so we used a more stringent E-value cutoff of 10<sup>−3</sup> to narrow in on top hits, identifying 924 structural homologs.</p><blockquote id="nnr7lo9cqxl"><p id="nxlxawcnde2">Download a <a href="https://zenodo.org/records/12775362/files/Foldseek_capsid_followup_05092024.xlsx?download=1" target="_blank">CSV</a> containing <strong>results from searching putatively endogenized viral capsid proteins against AlphaFold</strong>/UniProt50 (version 4), AlphaFold/Swiss-Prot (version 4), and AlphaFold/Proteome (version 4) using the Foldseek web server.</p></blockquote><h2 id="structural-clustering-to-identify-capsids-in-ornithodoros-turicata">Structural clustering to identify capsids in <em>Ornithodoros turicata</em></h2><p id="n3eyr8hnf52">We tested a different approach to identify capsids that have been endogenized by ticks using structure-based clustering and the ProteinCartography pipeline <span id="nse7oxmjhwh" data-node-type="citation" data-value="https://doi.org/10.57844/arcadia-a5a6-1068" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="nse7oxmjhwh-note-popover" contenteditable="false">[17]</span>. We co-clustered viral proteins with tick proteins and identified clusters where tick proteins co-cluster with known viral capsids.</p><p id="nvvhgctdio1">For this analysis, we first needed to obtain virus capsid protein structures for comparison. As stated previously, the AlphaFold database doesn't currently include viral proteins. Therefore, we performed structural predictions using ESMFold for all capsid-related proteins from the VOG database. We chose to use ESMFold in this case because it’s faster than AlphaFold, which was important for the number of proteins we needed to fold <span id="nn9efphrjyq" data-node-type="citation" data-value="https://doi.org/10.1126/science.ade2574" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="nn9efphrjyq-note-popover" contenteditable="false">[18]</span>.</p><blockquote id="nhda8y7pwe0"><p id="nd8ex9mpv4c">These <strong>predicted capsid protein structures</strong> are on <a href="https://zenodo.org/records/12796464" target="_blank">Zenodo</a> in the “<a href="https://zenodo.org/records/12796464/files/structures.zip?download=1" target="_blank">structures.zip</a>” file.</p></blockquote><p id="n8uc28o4mgm">We searched UniProt for all proteins from <em>Ornithodoros turicata </em>(UniProt organism ID: 34597)<em>. </em>We chose to start with this particular tick because there are a tractable number (&lt; 10,000) of structures from <em>O. turicata </em>on UniProt, and many of these structures were predicted from its<em> </em>salivary transcriptome. We then downloaded the associated metadata and structures using scripts from the ProteinCartography repo <span id="nurxhl9nsam" data-node-type="citation" data-value="https://doi.org/10.57844/arcadia-a5a6-1068" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="nurxhl9nsam-note-popover" contenteditable="false">[17]</span>.</p><blockquote id="nyzog6j6l8a"><p id="ngos3gi9js2"><strong>Python scripts</strong> for downloading metadata are <a href="https://github.com/Arcadia-Science/ProteinCartography/blob/v0.4.2/ProteinCartography/fetch_uniprot_metadata.py" target="_blank">here</a>, the <strong>script</strong> for downloading structures is <a href="https://github.com/Arcadia-Science/ProteinCartography/blob/v0.4.2/ProteinCartography/download_pdbs.py" target="_blank">here</a>, and <strong>notebooks</strong> to facilitate metadata and structure downloads are in <a href="https://github.com/Arcadia-Science/capsids/tree/pubv1.0" target="_blank">this GitHub repo</a>.</p></blockquote><p id="n7fflv8pw07">To test this co-clustering approach, we performed ProteinCartography clustering analyses on combinations of viral structures and tick structures. This run contained the entire collection of capsid-related VOG proteins and all of the tick proteins from <em>O. turicata</em>. ProteinCartography output files include interactive UMAP and t-SNE embeddings with metadata overlays. Also included in ProteinCartography’s outputs are the corresponding TSV files that contain plotting information and all the metadata for each analysis, an interactive heatmap showing the similarity within and between clusters, and all the intermediate outputs. We looked for structural clusters in which viral proteins co-clustered with tick proteins. We found that the <em>Ornithodoros </em>proteins A0A2R5L5R2 and A0A2R5L7H1 cluster with proteins from VOG00029, VOG05312, VOG06972, and VOG20608.&nbsp;</p><blockquote id="nif85i15ci2"><p id="ntnoa9d8xbw">The <strong>data</strong> we used for this analysis are on <a href="https://zenodo.org/records/12796464" target="_blank">Zenodo</a>, the <strong>code</strong> is in <a href="https://github.com/Arcadia-Science/ProteinCartography/tree/v0.4.2" target="_blank">this GitHub repo</a>, and <strong>additional notebooks</strong> to facilitate metadata and structure downloading and custom plotting are in <a href="https://github.com/Arcadia-Science/capsids" target="_blank">this GitHub repo</a>.</p></blockquote><h2 id="taxonomic-tree-visualizations">Taxonomic tree visualizations&nbsp;</h2><p id="nf0hhesxz82">We used the scientific species names from each analysis to create a taxonomic tree using phyloT (version 2) and the phyloT database (version 2023.2). Using the <a href="https://phylot.biobyte.de/index.cgi" target="_blank">phyloT web server</a>, we set the node identifiers to scientific names, used expanded internal nodes, set polytomy to yes, and exported the trees as Newick files that we visualized and further annotated in iTOL (version 6.9) <span id="nt92ou9mx66" data-node-type="citation" data-value="https://doi.org/10.1093/nar/gkab301" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="nt92ou9mx66-note-popover" contenteditable="false">[19]</span>.&nbsp;</p><blockquote id="n1k75ue38ff"><p id="nvs4zvadynb"><strong>Newick files for the trees</strong> in <a class="reference" data-node-type="reference" data-target-id="neezumhdcet" href="#neezumhdcet" contenteditable="false">Figure 3</a> and <a class="reference" data-node-type="reference" data-target-id="n41kwh0e03y" href="#n41kwh0e03y" contenteditable="false">Figure 5</a> are available on <a href="https://zenodo.org/records/12775362" target="_blank">Zenodo</a>.</p></blockquote><p id="ndxh3attl17"><br class="ProseMirror-trailingBreak"></p><blockquote id="nylyp0om911"><p id="nvmlfrjpo4q">All <strong>code</strong> generated and used for the pub is available in this GitHub repository (DOI: <a href="https://doi.org/10.5281/zenodo.12809305" target="_blank">10.5281/zenodo.12809305</a>), including scripts for performing HMM and BLAST searches, plus notebooks for ProteinCartography analyses.</p></blockquote><h1 id="the-results">The results</h1><blockquote id="nzu7la48srk"><p id="nb9k1ub5poc"><strong>SHOW ME THE DATA</strong>: Access data from our <strong>capsid HMM search</strong>, our <strong>preHGT run</strong> to find putatively endogenized viral capsids, and our <strong>follow-up work</strong> looking into individual hits with BLAST searches of tick salivary transcriptome, genomic neighborhood analysis, and Foldseek analysis on <a href="https://zenodo.org/records/12775362" target="_blank">Zenodo</a> (DOI: <a href="https://doi.org/10.5281/zenodo.12775362" target="_blank">10.5281/zenodo.12775362</a>).</p></blockquote><p id="ntmsmcrxhh5"><br class="ProseMirror-trailingBreak"></p><blockquote id="noyli7r1c42"><p id="nn18t2ee9gh"><strong>Data</strong> from our ProteinCartography runs, including structures of all tick and viral capsid proteins, the configuration file, and all outputs, are in a separate <a href="https://zenodo.org/records/12796464" target="_blank">Zenodo</a> record (DOI: <a href="https://doi.org/10.5281/zenodo.12796464" target="_blank">10.5281/zenodo.12796464</a>).</p></blockquote><h2 id="sequence-based-searches-highlight-retrotransposon-capsid-like-genes-in-parasitoid-wasps-and-other-parasitic-species-including-ticks">Sequence-based searches highlight retrotransposon capsid-like genes in parasitoid wasps and other parasitic species, including ticks</h2><p id="nev108k8217">We first used a sequence-similarity approach to search for capsid-like sequences in venomous species. Because there's a large body of public data collected from venom samples, we were able to curate a database of 156 venom transcriptomes from 135 different species <span id="nfpdu1sa3u9" data-node-type="citation" data-value="https://doi.org/10.57844/arcadia-d3dc-7499" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="nfpdu1sa3u9-note-popover" contenteditable="false">[20]</span>. We searched this database using viral capsid-specific hidden Markov model (HMM) profiles from the VOG database. Because the bracovirus capsid isn't represented in the VOG database, we also generated a custom bracovirus capsid HMM to use in searching against the venom transcriptomes.</p><p id="n2jgc6tbah0">Searches using VOG capsid HMMs generated over 3,000 hits before filtering. However, the bracovirus vp39 capsid HMM only hit the <em>Microplitis demolitor</em> transcriptome and didn’t have any other high-confidence hits. We conclude that bracovirus-like capsids are not broadly present across animal venoms. That said, this transcriptome-based search likely underrepresents the true distribution of domesticated bracovirus capsids within parasitoid wasps. Previous work on bracoviruses shows that they’re primarily present in the calyx fluid from wasp ovaries, which is injected during oviposition along with venom gland secretions. While the <em>Microplitus demolitor</em> transcriptome we used came from both the wasp venom gland and ovaries, the other parasitoid data we used only included transcripts from venom glands <span id="nvfxy2strze" data-node-type="citation" data-value="https://doi.org/10.1038/s42003-020-01623-8" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="nvfxy2strze-note-popover" contenteditable="false">[4]</span>. The lack of ovary transcriptome data may, in part, explain the absence of bracovirus sequences from those parasitoid wasp transcriptomes.</p><p id="ns478ydyosi">We performed manual curation of the top 210 VOG HMM hits via BLAST, which suggested that many of the hits were false positives. Some false positives came from potential contaminating or venom virome-associated sequences, including bacteriophage capsid proteins. Other false positives came from hits of our viral HMMs to shared domains in eukaryotic genes that are probably not derived from recent viral capture and endogenization. Many of our false positive hits had immunoglobulin domains or protease domains, which are common to both viral proteins and host proteins.&nbsp;</p><p id="nsxjv70kcjk">These filters limited our homologs of interest to 14 total hits (not including the vp39 bracovirus capsid hit in <em>Microplitis demolitor) </em>(<a class="reference" data-node-type="reference" data-target-id="nheciqbgp4q" href="#nheciqbgp4q" contenteditable="false">Figure 2</a>). These viral hits came from four species of parasitoid wasp (<em>Cotesia vestalis</em>, <em>Diadromus collaris</em>, <em>Microplitis demolitor</em>, and <em>Pteromalus puparum</em>), and one spider (<em>Phoneutria nigriventer</em>). We were initially disappointed in the results from this search because it mostly returned endogenized proteins in wasps, which are already known to have domesticated capsids. Our goal had been to identify endogenized capsid proteins outside of parasitoids, ideally in species of venomous animals that bite humans or other mammals. However, when examining the taxonomic distribution of the 14 hits using BLAST, we found that one of these proteins — a Gag-Pol protein from LTR-type retrotransposons — had homologs outside of parasitoid wasps, including in ticks, which bite and feed on humans.</p><figure id="nheciqbgp4q" data-node-type="image" data-size="85" data-align="center" data-url="https://assets.pubpub.org/14a1n879/Fig2_Venomous-sequence-based-01721329919003.png" data-caption="<p id=&quot;nza6ycxm2c2&quot; data-reactroot=&quot;&quot;><strong>A sequence-based search of 135 species’ transcriptomes identifies putatively endogenized viral capsids expressed in venom</strong>.</p><p id=&quot;r2939193228&quot; data-reactroot=&quot;&quot;>We searched venom transcriptomes from 135 species, which included venomous snakes, an annelid, mollusks, arachnids, centipedes, a crustacean, and parasitoid wasps. These species are displayed in a taxonomic tree. We did not find evidence of endogenized capsids in the venoms of most animals, indicated with black circles on leaf tips. However, four species of parasitoid wasp (<em>Cotesia vestalis</em>, <em>Diadromus collaris</em>, <em>Microplitis demolitor,</em> and<em> Pteromalus puparum</em>) have putatively endogenized capsids in their venoms, as does the Brazilian wandering spider (<em>Phoneutria nigriventer</em>). These instances of potential domestication are highlighted with red circles. Individual species names are omitted from this figure due to the size and complexity of the tree, but the associated Newick file can be found <a href=&quot;https://zenodo.org/records/12775362/files/Fig2_Newick_HMM_venoms.txt?download=1&quot; target=&quot;_blank&quot;>here</a>, the list of input transcriptomes is <a href=&quot;https://github.com/Arcadia-Science/capsids/blob/main/metadata/SRA_TSA_venom_gland_accessions.csv&quot; target=&quot;_blank&quot;>here</a>, and the hits from this sequence-based search are <a href=&quot;https://zenodo.org/records/12775362/files/Annotated_capsid_no_cutoffs_summary_with_seqs_05092024.xlsx?download=1&quot; target=&quot;_blank&quot;>here</a>.</p>" data-alt-text="" contenteditable="false"><img srcset="https://resize-v3.pubpub.org/eyJidWNrZXQiOiJhc3NldHMucHVicHViLm9yZyIsImtleSI6IjE0YTFuODc5L0ZpZzJfVmVub21vdXMtc2VxdWVuY2UtYmFzZWQtMDE3MjEzMjk5MTkwMDMucG5nIiwiZWRpdHMiOnsicmVzaXplIjp7IndpZHRoIjo4MDAsImZpdCI6Imluc2lkZSIsIndpdGhvdXRFbmxhcmdlbWVudCI6dHJ1ZX19fQ== 1x,https://resize-v3.pubpub.org/eyJidWNrZXQiOiJhc3NldHMucHVicHViLm9yZyIsImtleSI6IjE0YTFuODc5L0ZpZzJfVmVub21vdXMtc2VxdWVuY2UtYmFzZWQtMDE3MjEzMjk5MTkwMDMucG5nIiwiZWRpdHMiOnsicmVzaXplIjp7IndpZHRoIjoxNjAwLCJmaXQiOiJpbnNpZGUiLCJ3aXRob3V0RW5sYXJnZW1lbnQiOnRydWV9fX0= 2x,https://resize-v3.pubpub.org/eyJidWNrZXQiOiJhc3NldHMucHVicHViLm9yZyIsImtleSI6IjE0YTFuODc5L0ZpZzJfVmVub21vdXMtc2VxdWVuY2UtYmFzZWQtMDE3MjEzMjk5MTkwMDMucG5nIiwiZWRpdHMiOnsicmVzaXplIjp7IndpZHRoIjoyNDAwLCJmaXQiOiJpbnNpZGUiLCJ3aXRob3V0RW5sYXJnZW1lbnQiOnRydWV9fX0= 3x" src="https://resize-v3.pubpub.org/eyJidWNrZXQiOiJhc3NldHMucHVicHViLm9yZyIsImtleSI6IjE0YTFuODc5L0ZpZzJfVmVub21vdXMtc2VxdWVuY2UtYmFzZWQtMDE3MjEzMjk5MTkwMDMucG5nIiwiZWRpdHMiOnsicmVzaXplIjp7IndpZHRoIjo4MDAsImZpdCI6Imluc2lkZSIsIndpdGhvdXRFbmxhcmdlbWVudCI6dHJ1ZX19fQ==" alt="" aria-labelledby="nheciqbgp4q-figure-caption"><figcaption id="nheciqbgp4q-figure-caption"><div><strong spellcheck="false">Figure 2</strong><div><p id="nza6ycxm2c2" data-reactroot=""><strong>A sequence-based search of 135 species’ transcriptomes identifies putatively endogenized viral capsids expressed in venom</strong>.</p><p id="r2939193228" data-reactroot="">We searched venom transcriptomes from 135 species, which included venomous snakes, an annelid, mollusks, arachnids, centipedes, a crustacean, and parasitoid wasps. These species are displayed in a taxonomic tree. We did not find evidence of endogenized capsids in the venoms of most animals, indicated with black circles on leaf tips. However, four species of parasitoid wasp (<em>Cotesia vestalis</em>, <em>Diadromus collaris</em>, <em>Microplitis demolitor,</em> and<em> Pteromalus puparum</em>) have putatively endogenized capsids in their venoms, as does the Brazilian wandering spider (<em>Phoneutria nigriventer</em>). These instances of potential domestication are highlighted with red circles. Individual species names are omitted from this figure due to the size and complexity of the tree, but the associated Newick file can be found <a href="https://zenodo.org/records/12775362/files/Fig2_Newick_HMM_venoms.txt?download=1" target="_blank">here</a>, the list of input transcriptomes is <a href="https://github.com/Arcadia-Science/capsids/blob/main/metadata/SRA_TSA_venom_gland_accessions.csv" target="_blank">here</a>, and the hits from this sequence-based search are <a href="https://zenodo.org/records/12775362/files/Annotated_capsid_no_cutoffs_summary_with_seqs_05092024.xlsx?download=1" target="_blank">here</a>.</p></div></div></figcaption></figure><p id="nx0rsasguuw">Interestingly, retrotransposon-derived Gag proteins form capsid-like structures, also known as viral-like particles (VLPs) <span id="n21s1zdw4e9" data-node-type="citation" data-value="https://doi.org/10.1186/gb-2004-5-6-225" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="n21s1zdw4e9-note-popover" contenteditable="false">[21]</span>. Further, Gag-Pol homologs from humans (Peg10) and <em>Drosophila</em> (Arc) were recently shown to be involved in RNA delivery between cells <span id="nr4qchkzcbv" data-node-type="citation" data-value="https://doi.org/10.1016/j.cell.2017.12.024" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="nr4qchkzcbv-note-popover" contenteditable="false">[22]</span><span id="n423klldoz7" data-node-type="citation" data-value="https://doi.org/10.1126/science.abg6155" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="n423klldoz7-note-popover" contenteditable="false">[23]</span>. We found that our wasp Gag-Pol protein of interest had homologs in parasitic species, including aphids, mosquitoes, and ticks. Of these species, we were particularly interested in the presence of capsid-like proteins in ticks, given our previous work to develop resources for studying the host-manipulating properties of tick saliva <span id="n6omh0jbdso" data-node-type="citation" data-value="https://doi.org/10.57844/arcadia-3hyh-3h83" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="n6omh0jbdso-note-popover" contenteditable="false">[24]</span>. Given that the salivary glands of ticks secrete venom protein homologs into their host upon biting, ticks can be considered venomous species <span id="nsc64ntpe52" data-node-type="citation" data-value="https://doi.org/10.1186/1742-9994-11-47" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="nsc64ntpe52-note-popover" contenteditable="false">[25]</span>. For these reasons, we prioritized one of the tick Gag protein homologs for further analysis as part of our investigation into capsid-like proteins in venoms (XP_042148722.1 from <em>Ixodes scapularis</em>) (Table 1).</p><h2 id="evidence-of-capsid-genes-horizontally-transferred-from-viruses-to-ixodes-tick-species">Evidence of capsid genes horizontally transferred from viruses to <em>Ixodes</em> tick species</h2><p id="nfwsry7s12l">Because the evolutionary acquisition of viral genes such as capsids is an example of inter-kingdom horizontal gene transfer (HGT), we next deployed a software pipeline specifically tailored to detect inter-kingdom HGT events — preHGT <span id="npibqhf7g9p" data-node-type="citation" data-value="https://doi.org/10.57844/arcadia-jfbp-7p11" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="npibqhf7g9p-note-popover" contenteditable="false">[26]</span>. Of the original venomous species included in our list, only nine had gene models available through NCBI (a prerequisite for preHGT) at the time we ran the pipeline. Although the original venomous species dataset we used for HMM-based analyses didn’t include tick species, we decided to include the ticks <em>Ixodes scapularis</em> and <em>Ixodes persulcatus</em> in our HGT analysis because we’d found a capsid protein in ticks in our follow-up BLAST searches, described above.</p><p id="njawn8k5vje">The preHGT analysis revealed 66 potential horizontal transfer events between our target species and all available genomes on NCBI. Of those, there were 21 instances of putative gene transfer from viruses to our species of interest. 14 of these were to parasitoid wasps, 11 of which were the previously studied bracovirus found within the <em>Microplitisdemolitor </em>genome. We identified seven viral HGT events outside of wasps, six of which we found in ticks (Table 1). The other non-wasp putative viral HGT event was of a nucleoside transporter in <em>Sepia pharaonis</em>, the pharaoh cuttlefish.</p><figure id="neezumhdcet" data-node-type="image" data-size="100" data-align="center" data-url="https://assets.pubpub.org/xcr9lti7/Fig3_Venomous-HGT-based-61721330313743.png" data-caption="<p id=&quot;n0dyswhi6ao&quot; data-reactroot=&quot;&quot;><strong>An HGT-based search reveals putative viral HGT events in the genomes of venomous animals</strong>.</p><p id=&quot;r5380540706&quot; data-reactroot=&quot;&quot;>We used the horizontal gene transfer (HGT) detection tool “preHGT” to identify putative viral HGT events in the genomes of nine venomous animals, including snakes, a cuttlefish, ticks, and parasitoid wasps. Evolutionary relationships between the species we searched are shown on a tree. We identified putative viral HGT events in ticks, parasitoid wasps, and the pharaoh cuttlefish. These events are marked by red circles at the leaf tips. The Newick file we used to generate this visual can be found <a href=&quot;https://zenodo.org/records/12775362/files/Fig3_Newick_HGT_genomes.txt?download=1&quot; target=&quot;_blank&quot;>here</a>, the list of input genomes <a href=&quot;https://zenodo.org/records/12775362/files/preHGT_input_genomes.csv?download=1&quot; target=&quot;_blank&quot;>here</a>, and the hits from this HGT search <a href=&quot;https://zenodo.org/records/12775362/files/Annotated_hgt_results_venom_050924.xlsx?download=1&quot;>here</a>.</p>" data-alt-text="" contenteditable="false"><img srcset="" src="https://assets.pubpub.org/xcr9lti7/Fig3_Venomous-HGT-based-61721330313743.png" alt="" aria-labelledby="neezumhdcet-figure-caption"><figcaption id="neezumhdcet-figure-caption"><div><strong spellcheck="false">Figure 3</strong><div><p id="n0dyswhi6ao" data-reactroot=""><strong>An HGT-based search reveals putative viral HGT events in the genomes of venomous animals</strong>.</p><p id="r5380540706" data-reactroot="">We used the horizontal gene transfer (HGT) detection tool “preHGT” to identify putative viral HGT events in the genomes of nine venomous animals, including snakes, a cuttlefish, ticks, and parasitoid wasps. Evolutionary relationships between the species we searched are shown on a tree. We identified putative viral HGT events in ticks, parasitoid wasps, and the pharaoh cuttlefish. These events are marked by red circles at the leaf tips. The Newick file we used to generate this visual can be found <a href="https://zenodo.org/records/12775362/files/Fig3_Newick_HGT_genomes.txt?download=1" target="_blank">here</a>, the list of input genomes <a href="https://zenodo.org/records/12775362/files/preHGT_input_genomes.csv?download=1" target="_blank">here</a>, and the hits from this HGT search <a href="https://zenodo.org/records/12775362/files/Annotated_hgt_results_venom_050924.xlsx?download=1">here</a>.</p></div></div></figcaption></figure><p id="nko51phff0i">Four of the six tick viral HGT events were predicted to be capsid-related sequences. These were all annotated as nucleocapsid proteins, or “N proteins,” from several different negative-stranded RNA viruses. These RNA viruses belong to groups known to infect ticks, such as Bunyavirales and Rhabdoviridae, yet these types of viruses don't go through a DNA phase in their replication. Integrated RNA viral genes have previously been noted in tick genomes, though their biological significance is unknown <span id="nghzon5btk9" data-node-type="citation" data-value="10.1101/2022.08.17.504163" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="nghzon5btk9-note-popover" contenteditable="false">[27]</span>. It’s intriguing to see RNA-derived viral nucleocapsids integrated into DNA-based tick genomes. The two other virus-to-tick HGT events in the dataset were both viral RNA-dependent RNA polymerase (RdRp). Nucleocapsid proteins from these RNA viruses are involved in binding and encapsulating RNA virus genomes and interact with viral RdRp to initiate viral genome replication <span id="nijybifkjkq" data-node-type="citation" data-value="https://doi.org/10.1002/wrna.1326" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="nijybifkjkq-note-popover" contenteditable="false">[28]</span><span id="nzr12hb21mm" data-node-type="citation" data-value="https://doi.org/10.3390/v12080835" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="nzr12hb21mm-note-popover" contenteditable="false">[29]</span>. Given these interesting findings, we decided to prioritize all of these additional tick proteins for further analysis (Table 1).</p><div class="tableWrapper" id="nvyypipzwot" data-suggestion-kind="null" data-suggestion-id="null" data-suggestion-user-id="null" data-suggestion-timestamp="null" data-suggestion-discussion-id="null" data-suggestion-original-attrs="null" data-align="null" data-size="null" data-smaller-font="false"><table style="width: 918px;"><colgroup><col style="width: 87px;"><col style="width: 109px;"><col style="width: 123px;"><col style="width: 109px;"><col style="width: 163px;"><col style="width: 111px;"><col style="width: 107px;"><col style="width: 109px;"></colgroup><tbody><tr><th data-colwidth="87"><p id="n9ylrxherq9"><strong><sup>Protein</sup></strong></p></th><th data-colwidth="109"><p id="n194773cs5v"><strong><sup>Tick species</sup></strong></p></th><th data-colwidth="123"><p id="ntpvx5669se"><strong><sup>Closest viral hit</sup></strong></p></th><th data-colwidth="109"><p id="nxambrzd5x8"><strong><sup>Annotation</sup></strong></p></th><th data-colwidth="163"><p id="n6sq6ycdli7"><strong><sup>Protein family</sup></strong></p></th><th data-colwidth="111"><p id="nadgu0x23sc"><strong><sup>Contains introns?</sup></strong></p></th><th data-colwidth="107"><p id="n1gtahokxmq"><strong><sup>Expressed in salivary gland?</sup></strong></p></th><th data-colwidth="109"><p id="npbnbd5ksxv"><strong><sup>Run through Foldseek?</sup></strong></p></th></tr><tr><td data-colwidth="87"><p id="nhrfr0zp7p9"><sup>XP_042148722.1</sup></p></td><td data-colwidth="109"><p id="nzo5faf34qx"><em><sup>Ixodes scapularis</sup></em></p></td><td data-colwidth="123"><p id="nd2mzbkung4"><sup>N/A</sup></p></td><td data-colwidth="109"><p id="n4p8itb4y43"><sup>Retro-transposon Gag protein</sup></p></td><td data-colwidth="163"><p id="nbyjofeb93l"><sup>pfam03732: Retrotrans_gag; pfam13650:Asp_protease_2; pfam00098; zf-CCHC&nbsp;</sup></p></td><td data-colwidth="111"><p id="nsb4gyqm4lt"><sup>N</sup></p></td><td data-colwidth="107"><p id="nsl7jls3swp"><sup>Y</sup></p></td><td data-colwidth="109"><p id="nfmp6dbmaj1"><sup>Y</sup></p><p id="n6y44oris81"><sup>(</sup><a class="reference" data-node-type="reference" data-target-id="n41kwh0e03y" href="#n41kwh0e03y" contenteditable="false">Figure 5</a><sup>, C)</sup></p></td></tr><tr><td data-colwidth="87"><p id="nz7i94gkqcv"><sup>KAG0420414.1</sup></p></td><td data-colwidth="109"><p id="n90j282kp5k"><em><sup>Ixodes persulcatus</sup></em></p></td><td data-colwidth="123"><p id="ncal30oeqar"><sup>Tahe rhabdovirus</sup></p></td><td data-colwidth="109"><p id="ngbuy06ba7g"><sup>N protein</sup></p></td><td data-colwidth="163"><p id="nmpa8om0wj1"><sup>pfam00945: Rhabdo_ncap</sup></p></td><td data-colwidth="111"><p id="nz2mbq0msr0"><sup>Y</sup></p></td><td data-colwidth="107"><p id="nr6y8n5zg8b"><sup>Y</sup></p></td><td data-colwidth="109"><p id="n43t69dp63i"><sup>N</sup></p></td></tr><tr><td data-colwidth="87"><p id="nh92emy7o8q"><sup>KAG0427517.1</sup></p></td><td data-colwidth="109"><p id="nu7jwh8b83x"><em><sup>Ixodes persulcatus</sup></em></p></td><td data-colwidth="123"><p id="nnqwbty40gh"><sup>Tahe rhabdovirus 3</sup></p></td><td data-colwidth="109"><p id="n3do866spgg"><sup>N protein</sup></p></td><td data-colwidth="163"><p id="npzhx3gamjh"><sup>pfam00945: Rhabdo_ncap</sup></p></td><td data-colwidth="111"><p id="n31rqowvey2"><sup>Y</sup></p></td><td data-colwidth="107"><p id="nfc0lo4szam"><sup>Y</sup></p></td><td data-colwidth="109"><p id="nckn9n95pro"><sup>Y</sup></p><p id="naay5eoi6t4"><sup>(</sup><a class="reference" data-node-type="reference" data-target-id="n41kwh0e03y" href="#n41kwh0e03y" contenteditable="false">Figure 5</a><sup>, B)</sup></p></td></tr><tr><td data-colwidth="87"><p id="nma28vvnqvs"><sup>EEC12039.1</sup></p></td><td data-colwidth="109"><p id="nl9eequ4u47"><em><sup>Ixodes scapularis</sup></em></p></td><td data-colwidth="123"><p id="nk913pc9p1s"><sup>Sara tick phlebovirus</sup></p></td><td data-colwidth="109"><p id="nz5qs58ddz6"><sup>N protein</sup></p></td><td data-colwidth="163"><p id="nk19p43vpo2"><sup>pfam05733: Tenui_N</sup></p></td><td data-colwidth="111"><p id="n5c1qs06brf"><sup>Y (gene is fragmented)</sup></p></td><td data-colwidth="107"><p id="n7c4tsu1s1m"><sup>N</sup></p></td><td data-colwidth="109"><p id="nu28j251qbu"><sup>N</sup></p></td></tr><tr><td data-colwidth="87"><p id="nepvvadswnx"><sup>EEC17452.1</sup></p></td><td data-colwidth="109"><p id="noive95xep5"><em><sup>Ixodes scapularis</sup></em></p></td><td data-colwidth="123"><p id="nsrj6ki35pb"><sup>South Bay sabavirus (bunyavirus order)&nbsp;</sup></p></td><td data-colwidth="109"><p id="n89p3kgag6l"><sup>N protein</sup></p></td><td data-colwidth="163"><p id="n6s9dhzjpnx"><sup>pfam02477: Nairo_nucleo</sup></p></td><td data-colwidth="111"><p id="n0tll2uqais"><sup>Y (gene is fragmented)</sup></p></td><td data-colwidth="107"><p id="nvkqftxgmyx"><sup>Y</sup></p></td><td data-colwidth="109"><p id="nmgwy5ey96b"><sup>Y</sup></p><p id="nircv4s9r7e"><sup>(</sup><a class="reference" data-node-type="reference" data-target-id="n41kwh0e03y" href="#n41kwh0e03y" contenteditable="false">Figure 5</a><sup>, A)</sup></p></td></tr><tr><td data-colwidth="87"><p id="nk2vbmx779o"><sup>KAG0443635.1</sup></p></td><td data-colwidth="109"><p id="n3gswdm0hm4"><em><sup>Ixodes persulcatus</sup></em></p></td><td data-colwidth="123"><p id="nkjil42qcdw"><sup>Totiviridae sp.</sup></p></td><td data-colwidth="109"><p id="nrygl9qo75a"><sup>RdRp</sup></p></td><td data-colwidth="163"><p id="nwujxxut5ea"><sup>N/A</sup></p></td><td data-colwidth="111"><p id="n6wgm5qa334"><sup>Y</sup></p></td><td data-colwidth="107"><p id="nvjoaj2sn7g"><sup>Y</sup></p></td><td data-colwidth="109"><p id="nn4jakh1n92"><sup>N</sup></p></td></tr><tr><td data-colwidth="87"><p id="ng1gsjhbh6g"><sup>KAG0444350.1</sup></p></td><td data-colwidth="109"><p id="n32bj4w501w"><em><sup>Ixodes persulcatus</sup></em></p></td><td data-colwidth="123"><p id="nwq4mnn06du"><sup>Phenuiviridae</sup></p></td><td data-colwidth="109"><p id="nn7nyif68mx"><sup>RdRp</sup></p></td><td data-colwidth="163"><p id="nure8ikbqdg"><sup>pfam12603: Bunya_L_PA-C-like</sup></p></td><td data-colwidth="111"><p id="nv4h1spcbmx"><sup>Y</sup></p></td><td data-colwidth="107"><p id="n1qic73r6cc"><sup>N</sup></p></td><td data-colwidth="109"><p id="nmk34hwh8ps"><sup>N</sup></p></td></tr></tbody></table></div><p id="n4yajpnk4j2"><strong><sup>Table 1</sup></strong><sup>.</sup><strong><sup> Tick proteins of potential viral origin</sup></strong><sup>. </sup><br><sup>This table lists the seven tick proteins of potential viral origin that we found in this study. We identified the first protein on this list (XP_042148722.1) as a homolog of a viral capsid endogenized by a parasitoid wasp. We identified the other six proteins using HGT-based analysis of animal genomes. This table shows the protein accession number and associated metadata: the tick species in which we found the protein, the virus that the gene probably originated from, and the protein’s annotation and protein family. We’re also showing the results of our manual genomic context analysis of each gene, where we looked for signs of domestication and salivary expression. We assessed whether each gene is inserted into the tick genome, if it has introns, and if it’s expressed in the salivary gland. Last, we highlight the proteins for which we did follow-up analysis using Foldseek and note the associated figure numbers. Annotated GenBank files of the contigs for each protein are available </sup><a href="https://zenodo.org/records/12775362" target="_blank"><sup>here</sup></a><sup>.</sup></p><h2 id="capsid-genes-identified-in-ticks-contain-introns-and-are-expressed-in-salivary-glands">Capsid genes identified in ticks contain introns and are expressed in salivary glands</h2><p id="n8kr3x2ee41">The HMM- and HGT-based searches led us to identify two different types of potential viral capsids in tick genomes — retrotransposon Gag-Pol capsid-like proteins and negative-stranded RNA viral nucleocapsids and associated RdRps. We next examined the genomic context of each of these genes to look for further evidence of endogenization. All seven genes are located within larger tick genomic contigs, suggesting integration into the tick genome rather than viral contamination that was sequenced along with the tick genome. Six out of seven contained introns (<a class="reference" data-node-type="reference" data-target-id="nd2wtjuapb1" href="#nd2wtjuapb1" contenteditable="false">Figure 4</a>), which suggests that the genes have been present in the tick genomes long enough to acquire eukaryotic genome characteristics. This result adds confidence that these sequences aren't derived from viral contaminants in the original datasets but, rather, are endogenized viral genes.</p><figure id="nd2wtjuapb1" data-node-type="image" data-size="100" data-align="center" data-url="https://assets.pubpub.org/kfizci85/Fig4_Tick-genomes-11721330518580.png" data-caption="<p id=&quot;nzle2tyxaew&quot; data-reactroot=&quot;&quot;><strong>Genomic context of two viral capsid genes endogenized in tick genomes</strong>.</p><p id=&quot;r1992780397&quot; data-reactroot=&quot;&quot;>Genomic context of the genes encoding KAG0427517.1, the rhabdovirus-derived N protein in <em>Ixodes persulcatus</em>, and XP_042148722.1, the Gag-Pol protein in <em>Ixodes scapularis</em>. Annotations (determined by looking for related proteins using BLAST, HHpred, and Foldseek) are included below each coding region. Genes of tick origin are grey-blue and viral-derived genes are dark orange. Introns are pale orange bars. Full annotated GenBank files of these contigs can be found <a href=&quot;https://zenodo.org/records/12775362/files/XP_042148722.1_contig.gb?download=1&quot; target=&quot;_blank&quot;>here</a> and <a href=&quot;https://zenodo.org/records/12775362/files/KAG0427517.1_contig.gb?download=1&quot; target=&quot;_blank&quot;>here</a>.</p>" data-alt-text="" contenteditable="false"><img srcset="https://resize-v3.pubpub.org/eyJidWNrZXQiOiJhc3NldHMucHVicHViLm9yZyIsImtleSI6ImtmaXpjaTg1L0ZpZzRfVGljay1nZW5vbWVzLTExNzIxMzMwNTE4NTgwLnBuZyIsImVkaXRzIjp7InJlc2l6ZSI6eyJ3aWR0aCI6ODAwLCJmaXQiOiJpbnNpZGUiLCJ3aXRob3V0RW5sYXJnZW1lbnQiOnRydWV9fX0= 1x,https://resize-v3.pubpub.org/eyJidWNrZXQiOiJhc3NldHMucHVicHViLm9yZyIsImtleSI6ImtmaXpjaTg1L0ZpZzRfVGljay1nZW5vbWVzLTExNzIxMzMwNTE4NTgwLnBuZyIsImVkaXRzIjp7InJlc2l6ZSI6eyJ3aWR0aCI6MTYwMCwiZml0IjoiaW5zaWRlIiwid2l0aG91dEVubGFyZ2VtZW50Ijp0cnVlfX19 2x,https://resize-v3.pubpub.org/eyJidWNrZXQiOiJhc3NldHMucHVicHViLm9yZyIsImtleSI6ImtmaXpjaTg1L0ZpZzRfVGljay1nZW5vbWVzLTExNzIxMzMwNTE4NTgwLnBuZyIsImVkaXRzIjp7InJlc2l6ZSI6eyJ3aWR0aCI6MjQwMCwiZml0IjoiaW5zaWRlIiwid2l0aG91dEVubGFyZ2VtZW50Ijp0cnVlfX19 3x" src="https://resize-v3.pubpub.org/eyJidWNrZXQiOiJhc3NldHMucHVicHViLm9yZyIsImtleSI6ImtmaXpjaTg1L0ZpZzRfVGljay1nZW5vbWVzLTExNzIxMzMwNTE4NTgwLnBuZyIsImVkaXRzIjp7InJlc2l6ZSI6eyJ3aWR0aCI6ODAwLCJmaXQiOiJpbnNpZGUiLCJ3aXRob3V0RW5sYXJnZW1lbnQiOnRydWV9fX0=" alt="" aria-labelledby="nd2wtjuapb1-figure-caption"><figcaption id="nd2wtjuapb1-figure-caption"><div><strong spellcheck="false">Figure 4</strong><div><p id="nzle2tyxaew" data-reactroot=""><strong>Genomic context of two viral capsid genes endogenized in tick genomes</strong>.</p><p id="r1992780397" data-reactroot="">Genomic context of the genes encoding KAG0427517.1, the rhabdovirus-derived N protein in <em>Ixodes persulcatus</em>, and XP_042148722.1, the Gag-Pol protein in <em>Ixodes scapularis</em>. Annotations (determined by looking for related proteins using BLAST, HHpred, and Foldseek) are included below each coding region. Genes of tick origin are grey-blue and viral-derived genes are dark orange. Introns are pale orange bars. Full annotated GenBank files of these contigs can be found <a href="https://zenodo.org/records/12775362/files/XP_042148722.1_contig.gb?download=1" target="_blank">here</a> and <a href="https://zenodo.org/records/12775362/files/KAG0427517.1_contig.gb?download=1" target="_blank">here</a>.</p></div></div></figcaption></figure><p id="nnploj4fw6x">Because we found most of the viral capsid candidates using the HGT approach rather than the transcriptome analysis, we wondered whether these genes are expressed. We were particularly curious whether they’re expressed in tick saliva, as this would suggest that the genes are involved in venom-related functions. We found all of our candidate tick capsids in <em>Ixodes scapularis</em> and <em>Ixodes persulcatus</em>; however, there's no publicly available <em>I. persulcatus</em> salivary transcriptome. There are salivary transcriptomes from <em>Ixodes scapularis </em>and<em> Ixodes ricinus, </em>so we used those to BLAST the capsid candidates. We found hits to five of the seven candidates, all in the <em>I. ricinus</em> salivary transcriptome, suggesting these genes have homologs in <em>I.</em> <em>ricinus</em> that are actively produced in the salivary glands.</p><h2 id="searching-venoms-for-proteins-with-structures-similar-to-viral-capsids-identifies-capsid-like-proteins-in-multiple-blood-sucking-species-and-other-invertebrates">Searching venoms for proteins with structures similar to viral capsids identifies capsid-like proteins in multiple blood-sucking species and other invertebrates</h2><p id="nbbjf27l8db">Due to the rapid sequence evolution of viral genomes, sequence-based approaches can often fail to detect similarity between distantly evolutionarily related genes. However, protein structure can remain conserved even when sequences diverge dramatically <span id="n9a8owxrumz" data-node-type="citation" data-value="https://doi.org/10.1111/brv.12905" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="n9a8owxrumz-note-popover" contenteditable="false">[30]</span>. For this reason, we next turned to structure-based homology searches to explore whether viral capsids may be present in additional species.</p><p id="n8a8m06f97t">To search for structural homologs of capsid proteins in venoms, we first used Foldseek, a tool that enables structure-based searches for similar proteins <span id="nbda2j9zxgi" data-node-type="citation" data-value="https://doi.org/10.1038/s41587-023-01773-0" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="nbda2j9zxgi-note-popover" contenteditable="false">[13]</span>. We performed structure-based searches on three representative capsid structural proteins (not the RdRps) identified above — the <em>Ixodes </em>rhabdovirus nucleocapsid (KAG0427517), the <em>Ixodes </em>sabavirus nucleocapsid (EEC17452), and the <em>Ixodes</em> Gag-Pol protein (XP_042148722) —&nbsp;as well as the bracovirus major capsid protein from the parasitoid wasp <em>Microplitis demolitor</em> (vp39, NP_001401748).</p><blockquote id="nzvm6ibnm3c"><p id="ne6ufhxuu5y"><strong>Dealing with disorder in structure comparison</strong></p><p id="n4gfiojdh06">One thing to note is that these proteins all contain disordered regions, which complicates structural interpretation. We get an idea of the amount of disorder in a protein using the pLDDT (predicted local distance difference test), which is calculated by AlphaFold and gives an estimate of the amount of confidence for the structure on a per-residue basis <span id="nd1u8ea983s" data-node-type="citation" data-value="https://doi.org/10.1093/bioinformatics/btt473" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="nd1u8ea983s-note-popover" contenteditable="false">[31]</span>. To get an idea of the total amount of disorder for the protein, we average the per-residue pLDDT scores. Regions of proteins that are well-ordered will have a score between 90–100, regions with a score between 70–90 have some structure but have less confidence, regions between 50–70 are generally low-confidence, and regions with a pLDDT of less than 50 are likely disordered <span id="n0j2a9gj96f" data-node-type="citation" data-value="https://doi.org/10.1101/2021.09.26.461876" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="n0j2a9gj96f-note-popover" contenteditable="false">[32]</span>. KAG0427517 had an average pLDDT of 62; EEC17452 had an average pLDDT of 77; NP_001401748 had an average pLDDT of 72; and XP_042148722 had an average pLDDT of 78. Despite the disorder throughout these proteins, we did get hits that align well with the structured regions of the proteins.</p></blockquote><figure id="n41kwh0e03y" data-node-type="image" data-size="85" data-align="center" data-url="https://assets.pubpub.org/2tvwltfs/Fig5_Dist.-structural-homologs-71721330929243.png" data-caption="<p id=&quot;n9kw5z7t31e&quot; data-reactroot=&quot;&quot;><strong>Distribution of structural homologs of endogenized viral proteins from ticks across the AlphaFold database</strong>.</p><p id=&quot;nw63ly6vnca&quot; data-reactroot=&quot;&quot;>We used Foldseek to identify structural homologs of:</p><p id=&quot;npidir78l5x&quot; data-reactroot=&quot;&quot;>(A) the tick sabavirus N protein (EEC17452.1)</p><p id=&quot;npjjdnkwd8m&quot; data-reactroot=&quot;&quot;>(B) the tick rhabdovirus N protein (KAG0427517.1)</p><p id=&quot;n1cdldu58kp&quot; data-reactroot=&quot;&quot;>(C) the tick Gag-Pol protein (XP_042148722.1). In this panel, we’ve collapsed all non-animal clades.&nbsp;</p><p id=&quot;nz2ku8o7l0o&quot; data-reactroot=&quot;&quot;>We show evolutionary relationships between the species with Foldseek hits to each protein using phylogenetic trees. In this case, every species on the tree has at least one hit to the query tick protein. Clades of animals with parasitic or venomous lifestyles are indicated with brown highlights around leaf tips and name, whereas non-venomous and non-parasitic organisms are shown with white highlighting.</p><p id=&quot;nni9lfn8bl2&quot; data-reactroot=&quot;&quot;><br/>Newick files for trees shown in this figure can be found <a href=&quot;https://zenodo.org/records/12775362&quot; target=&quot;_blank&quot;>here</a>. In panel C, we couldn’t show individual species names due to the size and complexity of the tree. However, information about all hits for each query, including their species of origin, can be found <a href=&quot;https://zenodo.org/records/12775362/files/Foldseek_capsid_followup_05092024.xlsx?download=1&quot; target=&quot;_blank&quot;>here</a>.</p>" data-alt-text="" contenteditable="false"><img srcset="https://resize-v3.pubpub.org/eyJidWNrZXQiOiJhc3NldHMucHVicHViLm9yZyIsImtleSI6IjJ0dndsdGZzL0ZpZzVfRGlzdC4tc3RydWN0dXJhbC1ob21vbG9ncy03MTcyMTMzMDkyOTI0My5wbmciLCJlZGl0cyI6eyJyZXNpemUiOnsid2lkdGgiOjgwMCwiZml0IjoiaW5zaWRlIiwid2l0aG91dEVubGFyZ2VtZW50Ijp0cnVlfX19 1x,https://resize-v3.pubpub.org/eyJidWNrZXQiOiJhc3NldHMucHVicHViLm9yZyIsImtleSI6IjJ0dndsdGZzL0ZpZzVfRGlzdC4tc3RydWN0dXJhbC1ob21vbG9ncy03MTcyMTMzMDkyOTI0My5wbmciLCJlZGl0cyI6eyJyZXNpemUiOnsid2lkdGgiOjE2MDAsImZpdCI6Imluc2lkZSIsIndpdGhvdXRFbmxhcmdlbWVudCI6dHJ1ZX19fQ== 2x,https://resize-v3.pubpub.org/eyJidWNrZXQiOiJhc3NldHMucHVicHViLm9yZyIsImtleSI6IjJ0dndsdGZzL0ZpZzVfRGlzdC4tc3RydWN0dXJhbC1ob21vbG9ncy03MTcyMTMzMDkyOTI0My5wbmciLCJlZGl0cyI6eyJyZXNpemUiOnsid2lkdGgiOjI0MDAsImZpdCI6Imluc2lkZSIsIndpdGhvdXRFbmxhcmdlbWVudCI6dHJ1ZX19fQ== 3x" src="https://resize-v3.pubpub.org/eyJidWNrZXQiOiJhc3NldHMucHVicHViLm9yZyIsImtleSI6IjJ0dndsdGZzL0ZpZzVfRGlzdC4tc3RydWN0dXJhbC1ob21vbG9ncy03MTcyMTMzMDkyOTI0My5wbmciLCJlZGl0cyI6eyJyZXNpemUiOnsid2lkdGgiOjgwMCwiZml0IjoiaW5zaWRlIiwid2l0aG91dEVubGFyZ2VtZW50Ijp0cnVlfX19" alt="" aria-labelledby="n41kwh0e03y-figure-caption"><figcaption id="n41kwh0e03y-figure-caption"><div><strong spellcheck="false">Figure 5</strong><div><p id="n9kw5z7t31e" data-reactroot=""><strong>Distribution of structural homologs of endogenized viral proteins from ticks across the AlphaFold database</strong>.</p><p id="nw63ly6vnca" data-reactroot="">We used Foldseek to identify structural homologs of:</p><p id="npidir78l5x" data-reactroot="">(A) the tick sabavirus N protein (EEC17452.1)</p><p id="npjjdnkwd8m" data-reactroot="">(B) the tick rhabdovirus N protein (KAG0427517.1)</p><p id="n1cdldu58kp" data-reactroot="">(C) the tick Gag-Pol protein (XP_042148722.1). In this panel, we’ve collapsed all non-animal clades.&nbsp;</p><p id="nz2ku8o7l0o" data-reactroot="">We show evolutionary relationships between the species with Foldseek hits to each protein using phylogenetic trees. In this case, every species on the tree has at least one hit to the query tick protein. Clades of animals with parasitic or venomous lifestyles are indicated with brown highlights around leaf tips and name, whereas non-venomous and non-parasitic organisms are shown with white highlighting.</p><p id="nni9lfn8bl2" data-reactroot=""><br>Newick files for trees shown in this figure can be found <a href="https://zenodo.org/records/12775362" target="_blank">here</a>. In panel C, we couldn’t show individual species names due to the size and complexity of the tree. However, information about all hits for each query, including their species of origin, can be found <a href="https://zenodo.org/records/12775362/files/Foldseek_capsid_followup_05092024.xlsx?download=1" target="_blank">here</a>.</p></div></div></figcaption></figure><h3 id="sabavirus-n-hits-eec174521">Sabavirus N hits (EEC17452.1)</h3><p id="nb93w8xhopd">The <em>I. scapularis</em> nucleocapsid protein EEC17452.1 showed homology to six tick proteins and one spider protein (<a class="reference" data-node-type="reference" data-target-id="n41kwh0e03y" href="#n41kwh0e03y" contenteditable="false">Figure 5</a>, A). We found the original EEC17452.1 protein plus one additional <em>I. scapularis</em> hit. We also had hits in <em>I. ricinus</em>; the soft tick, <em>Ornithodros erraticus</em>; and the orb-weaver spider, <em>Araneus ventricosus.</em></p><h3 id="rhabdovirus-n-hits-kag04275171">Rhabdovirus N hits (KAG0427517.1)</h3><p id="ng6l037q1wb">We found structural homologs of the <em>Ixodes persulcatus </em>rhabdovirus nucleocapsid protein KAG0427517 protein in the ticks<em> Ixodes scapularis</em> and <em>Rhipicephalus microplus</em>, as well as several other species with intriguing parallels to ticks (<a class="reference" data-node-type="reference" data-target-id="n41kwh0e03y" href="#n41kwh0e03y" contenteditable="false">Figure 5</a>, B). Namely, we found structural homologs in other hematophagous (blood-sucking) species — mosquitos (<em>Aedes aegypti</em> and <em>Anopheles darlingi</em>) and bed bugs (<em>Cimex lectularius</em>). Intriguingly, we also found homologs in two sap-sucking species — the aphids <em>Aphis craccivora</em> and <em>Sipha flava</em>. We also found homologs in the plant parasitic spider mite,<em> Tetranychus urticae</em>, and the human parasitic helminth worm, <em>Enterobius vermicularis</em>, also known as pinworm. This pattern is compelling because all of these species, including ticks, have parasitic interactions with their host species, suggesting that the viral capsid may play a role in diverse types of parasitic lifestyles. The only non-parasites that had structural homologs of these proteins were the diamondback moth, <em>Plutella xylostella,</em> and the speckled wood butterfly, <em>Pararge aegeria</em>. Interestingly, <em>P. xylostella</em> is a natural host of several parasitoid wasps, suggesting that the nucleocapsid protein could be the result of parasite-to-host HGT. The significance of the N protein homolog in <em>P. aegeria </em>is unknown.</p><h3 id="gag-pol-hits-xp0421487221">Gag-Pol hits (XP_042148722.1)</h3><p id="nlihn7sj1v3">The tick retrotransposon Gag-Pol protein XP_042148722.1 returned 924 hits in our Foldseek search. These hits are widely distributed across eukaryotes, including many parasitic blood and sap-feeding species (<a class="reference" data-node-type="reference" data-target-id="n41kwh0e03y" href="#n41kwh0e03y" contenteditable="false">Figure 5</a>, C). This Foldseek search also identified Arc1 (3.69 × 10<sup>−8</sup>) and Arc2 (2.00 × 10<sup>−5</sup>) from <em>Drosophila melanogaster</em> as structural homologs of the tick gal-pol protein. Drosophila Arc proteins are domesticated viral capsids that participate in RNA delivery between cells <span id="nf1f1mqws1r" data-node-type="citation" data-value="https://doi.org/10.1016/j.cell.2017.12.024" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="nf1f1mqws1r-note-popover" contenteditable="false">[22]</span>.</p><h3 id="bracovirus-vp39-hits-np001401748">Bracovirus vp39 hits (NP_001401748)</h3><p id="nlko0k1q33z">The only structural homologs we found of the bracovirus vp39 protein NP_001401748 are from other parasitoid wasps, <em>Cotesia chilonis</em> and <em>Chelonus inanitus</em>.</p><h2 id="co-clustering-venom-proteins-with-known-viral-capsids-reveals-a-putative-uncharacterized-eukaryotic-viral-capsid">Co-clustering venom proteins with known viral capsids reveals a putative, uncharacterized eukaryotic viral capsid</h2><p id="naekhpd2fh3">We next wanted to see if we could extend structural homology-based approaches to whole-venom transcriptomes. Because all of our prior results highlighted the presence of potential capsids in tick species, we chose to focus on proteins from tick salivary transcriptomes for this analysis. We analyzed structures from <em>Ornithodoros turicata</em>, a species of soft-bodied tick in the family <em>Argasidae.</em> We aggregated folded protein structures from <em>O. turicata</em> and folded protein structures from known capsid proteins from the VOG database. We then performed clustering to see if any capsids would cluster with proteins from tick saliva. We identified 47 structural clusters (Figure 6, A). From this starting point, we could explore the candidate list of clusters for possible evidence of viral proteins clustering with venom proteins. We see the majority of clusters are quite specific either to viruses or to <em>O. turicata</em>, but there were six candidate clusters containing both a mix of proteins from the two datasets and a high degree of structural homology (TM &gt; 0.5).</p><figure id="nult3fgq8zx" data-node-type="image" data-size="100" data-align="center" data-url="https://assets.pubpub.org/l9sytd93/Fig6_PC-tick-proteins-51721331036541.png" data-caption="<p id=&quot;ncdnnipu0mu&quot; data-reactroot=&quot;&quot;><strong>ProteinCartography analysis of viral capsid proteins with </strong><strong><em>Ornithodoros turicata </em></strong><strong>proteins reveals two tick proteins that cluster with capsid proteins</strong>.</p><p id=&quot;r39331335&quot; data-reactroot=&quot;&quot;>We clustered all <em>O. turicata </em>structures available on UniProt with ESMFold predictions of the capsid proteins in the VOG database using ProteinCartography.</p><p id=&quot;r3421465495&quot; data-reactroot=&quot;&quot;>(A) A UMAP of co-clustered tick and viral proteins. Here, each protein is a single point on the UMAP and is colored by cluster membership.</p><p id=&quot;r7148496963&quot; data-reactroot=&quot;&quot;>(B) The same UMAP as in (A), but here tick proteins are green, and viral proteins are grey. Generally, the capsid proteins and tick proteins are clustered separately). We identified cluster 22 (boxed) as a cluster that contained two tick proteins alongside proteins from four different viral capsid VOGs.</p><p id=&quot;r3770214916&quot; data-reactroot=&quot;&quot;>(C) Zoom-in on cluster 22. Tick proteins A0A2R5L5R2 and A0A2R5L7H1 cluster with VOG proteins, some of which are annotated as mimivirus capsids.</p>" data-alt-text="" contenteditable="false"><img srcset="" src="https://assets.pubpub.org/l9sytd93/Fig6_PC-tick-proteins-51721331036541.png" alt="" aria-labelledby="nult3fgq8zx-figure-caption"><figcaption id="nult3fgq8zx-figure-caption"><div><strong spellcheck="false">Figure 6</strong><div><p id="ncdnnipu0mu" data-reactroot=""><strong>ProteinCartography analysis of viral capsid proteins with </strong><strong><em>Ornithodoros turicata </em></strong><strong>proteins reveals two tick proteins that cluster with capsid proteins</strong>.</p><p id="r39331335" data-reactroot="">We clustered all <em>O. turicata </em>structures available on UniProt with ESMFold predictions of the capsid proteins in the VOG database using ProteinCartography.</p><p id="r3421465495" data-reactroot="">(A) A UMAP of co-clustered tick and viral proteins. Here, each protein is a single point on the UMAP and is colored by cluster membership.</p><p id="r7148496963" data-reactroot="">(B) The same UMAP as in (A), but here tick proteins are green, and viral proteins are grey. Generally, the capsid proteins and tick proteins are clustered separately). We identified cluster 22 (boxed) as a cluster that contained two tick proteins alongside proteins from four different viral capsid VOGs.</p><p id="r3770214916" data-reactroot="">(C) Zoom-in on cluster 22. Tick proteins A0A2R5L5R2 and A0A2R5L7H1 cluster with VOG proteins, some of which are annotated as mimivirus capsids.</p></div></div></figcaption></figure><p id="nkpq5eobm90">We followed up on cluster 22, which contained tick proteins A0A2R5L5R2 and A0A2R5L7H1, along with proteins annotated as “major capsid protein” from mimivirus (<a class="reference" data-node-type="reference" data-target-id="nult3fgq8zx" href="#nult3fgq8zx" contenteditable="false">Figure 6</a>, B). Mimiviruses are giant viruses that typically infect species of amoeba <span id="ngbl4tmfhsq" data-node-type="citation" data-value="https://doi.org/10.1126/science.1081867" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="ngbl4tmfhsq-note-popover" contenteditable="false">[33]</span>. However, the mimivirus capsid protein is in the same protein family as the capsid from African swine fever virus, which <em>O. turicata</em> ticks transmit. By sequence, the <em>O. turicata</em> proteins found in this cluster are highly divergent from capsids from both the mimivirus and African swine fever virus. Unfortunately, there are no <em>O. turicata</em> genomes available, so it’s not currently possible to see if this protein comes from an unknown contaminating virus or is endogenous to the tick genome.</p><p id="nenv3q0wzfq">After our initial test with <em>O. turicata</em>, we didn't pursue this clustering approach further. This method is computationally challenging to scale outside of a few salivary transcriptomes, and our ability to interpret the hits that come out of this approach is limited.</p><h1 id="key-takeaways">Key takeaways</h1><ul id="nmd75x9j2xs"><li><p id="ni4utg7mbrf"><span class="discussion-range d-ea9e3be6-cf56-4360-9115-e74d31df7b3b">We detected putative endogenized viral capsids encoded by ticks and several other parasitic species, and the tick-encoded viral capsids are expressed in tick saliva. </span><span class="discussion-mount dm-ea9e3be6-cf56-4360-9115-e74d31df7b3b ProseMirror-widget" contenteditable="false" style="display: inline;"><span class="discussion-group-component" tabindex="-1" style="left: 830.172px; width: 292.336px;"><span class="discussion-nav-component" style="color: rgb(72, 75, 80); border-bottom: 1px solid transparent;"><style>.d-undefined, .lh-undefined { background-color: rgba(0, 0, 0, 0.2) !important; }</style><span class="discussion-bubble-component" role="button" tabindex="0"><span class="bubble" style="border: 1px solid rgb(72, 75, 80);">2</span><span class="label">Cyril Eleftheriou, Adair L. Borges</span></span></span></span></span><img class="ProseMirror-separator" alt=""><br class="ProseMirror-trailingBreak"></p></li><li><p id="nm6lvf7yhp7">Our highest-confidence tick-encoded capsid types are RNA nucleocapsids and retrotransposon Gag-Pol proteins. We predict that both interact with and help package RNA molecules.</p></li><li><p id="nh13etjnplo">These results suggest the possibility that some parasites use packaged nucleic acids to interact with their hosts.</p></li></ul><h1 id="discussion">Discussion</h1><p id="n4lx8zz6nob">Venomous species have evolved an enormous biological arsenal of host-manipulating factors. This particular study was inspired by the ancient use of viral capsids for host gene delivery by parasitoid wasps. We took a multi-pronged search across venomous species, looking for examples where other venomous species may have evolved the use of viral capsids. The two major categories of viral capsids we identified originate from RNA viruses [rhabdovirus and sabavirus (a type of bunyavirus)] and the retrotransposon Gag-Pol protein from LTR retrotransposons.</p><p id="nlfu4y6pc8y">We were excited to detect these capsid-like proteins outside of parasitoid wasps (<a class="reference" data-node-type="reference" data-target-id="njp74m066hd" href="#njp74m066hd" contenteditable="false">Figure 1</a>). Taxonomic analysis of these species suggests that some endogenized capsids are enriched in organisms that have parasitic interactions with their hosts. In particular, several tick species appear to have homologs of multiple types of viral capsids. These appear to have been endogenized in the ticks’ genomes and expressed in their saliva, the tick equivalent of venom. Integration of viral sequences into tick genomes has been observed before <span id="ne1hy6u8qbd" data-node-type="citation" data-value="https://doi.org/10.3389/fcimb.2013.00025" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="ne1hy6u8qbd-note-popover" contenteditable="false">[34]</span><span id="nk04jpan6h6" data-node-type="citation" data-value="https://doi.org/10.1093/ve/vez010" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="nk04jpan6h6-note-popover" contenteditable="false">[35]</span><span id="nhplnsy2ytb" data-node-type="citation" data-value="https://doi.org/10.1101/2022.08.17.504163" data-unstructured-value="" data-custom-label="" class="citation" tabindex="0" role="link" aria-describedby="nhplnsy2ytb-note-popover" contenteditable="false">[36]</span>. Hard ticks feed on hosts for days to weeks, unlike the rapid predator-prey interactions found in many venomous species, such as snakes. This extended host-parasite interaction could be analogous to the extended interaction of parasitoid wasps and lepidopteran hosts. This sustained interaction likely requires modes of host manipulation that act over longer timescales — gene delivery may be one of these.</p><p id="nn0ufcwtevs">Our findings suggest that the endogenization of viral capsids by venomous and/or parasitic species may be more widespread than previously understood, and these endogenized capsids are common in parasitic species that have extended interactions with their hosts. Right now, we don’t know the physiological role, if any, of these endogenized capsids in the parasitic lifestyle. However, we believe our findings are an intriguing starting point for further study into the specific roles of these capsids in host–parasite biology and may suggest novel approaches to molecular delivery in mammals.</p><h1 id="next-steps">Next steps</h1><p id="n42nnaxb0xj">Our initial work has established that capsid-like proteins are found in a variety of parasitic species, including some that bite humans. The biological significance of these endogenized capsids is still unknown. For us, the most translationally exciting outcome would be that capsids from human parasites are participating in gene delivery to human cells. Future work should focus on understanding whether any of these organisms actually use these capsids for nucleic acid delivery and, if so, what specific cargo they deliver.</p><p id="nihj4yai9qz"><br class="ProseMirror-trailingBreak"></p><div contenteditable="false"><hr></div><figure id="nkbbuug3q2z" data-node-type="iframe" data-size="100" data-align="center" contenteditable="false"><iframe alt="" src="https://publishing-tools.arcadiascience.com/typeform/AacF8YKM?pub_name=rec581s2eybk6I5Cw" height="425" aria-describedby="nkbbuug3q2z-figure-caption" allowfullscreen="true"></iframe><figcaption id="nkbbuug3q2z-figure-caption"><div><div></div></div></figcaption></figure><blockquote id="n4mp6p82h92"><p id="nq5xgpyt2xh">Share your thoughts!</p><p id="nz00o4ipzt9"><strong><a href="https://bit.ly/3xo7dkm" target="_blank">Watch a video tutorial</a></strong> on making a PubPub account and commenting. Feel free to add line-by-line comments anywhere within this text, provide overall feedback by commenting in the box at the bottom of the page, or post about this work on social media. Please make all feedback public so other readers can benefit from the discussion.</p></blockquote><figure id="nk936e7tnye" data-node-type="iframe" data-size="100" data-align="center" contenteditable="false"><iframe alt="" src="https://publishing-tools.arcadiascience.com/twitter/398034609045805608" height="500" aria-describedby="nk936e7tnye-figure-caption" allowfullscreen="true"></iframe><figcaption id="nk936e7tnye-figure-caption"><div><div></div></div></figcaption></figure><div contenteditable="false"><hr></div></div>`;
